// lucide-solid/dist/source/aliases/suffixed.jsx
/**
* @license lucide-solid v0.577.0 - ISC
*
* This source code is licensed under the ISC license.
* See the LICENSE file in the root directory of this source tree.
*/

// src/aliases/suffixed.ts
import { default as default2 } from "../icons/a-arrow-down";
import { default as default3 } from "../icons/a-arrow-up";
import { default as default4 } from "../icons/a-large-small";
import { default as default5 } from "../icons/accessibility";
import { default as default6 } from "../icons/activity";
import { default as default7 } from "../icons/air-vent";
import { default as default8 } from "../icons/airplay";
import { default as default9 } from "../icons/alarm-clock-off";
import { default as default10 } from "../icons/alarm-clock";
import { default as default11 } from "../icons/alarm-smoke";
import { default as default12 } from "../icons/album";
import { default as default13 } from "../icons/align-end-horizontal";
import { default as default14 } from "../icons/align-center-horizontal";
import { default as default15 } from "../icons/align-center-vertical";
import { default as default16 } from "../icons/align-horizontal-distribute-center";
import { default as default17 } from "../icons/align-end-vertical";
import { default as default18 } from "../icons/align-horizontal-distribute-end";
import { default as default19 } from "../icons/align-horizontal-justify-center";
import { default as default20 } from "../icons/align-horizontal-distribute-start";
import { default as default21 } from "../icons/align-horizontal-justify-end";
import { default as default22 } from "../icons/align-horizontal-justify-start";
import { default as default23 } from "../icons/align-horizontal-space-around";
import { default as default24 } from "../icons/align-horizontal-space-between";
import { default as default25 } from "../icons/align-start-vertical";
import { default as default26 } from "../icons/align-start-horizontal";
import { default as default27 } from "../icons/align-vertical-distribute-center";
import { default as default28 } from "../icons/align-vertical-distribute-end";
import { default as default29 } from "../icons/align-vertical-distribute-start";
import { default as default30 } from "../icons/align-vertical-justify-center";
import { default as default31 } from "../icons/align-vertical-justify-end";
import { default as default32 } from "../icons/align-vertical-justify-start";
import { default as default33 } from "../icons/align-vertical-space-around";
import { default as default34 } from "../icons/ambulance";
import { default as default35 } from "../icons/align-vertical-space-between";
import { default as default36 } from "../icons/ampersand";
import { default as default37 } from "../icons/ampersands";
import { default as default38 } from "../icons/anchor";
import { default as default39 } from "../icons/amphora";
import { default as default40 } from "../icons/angry";
import { default as default41 } from "../icons/annoyed";
import { default as default42 } from "../icons/anvil";
import { default as default43 } from "../icons/antenna";
import { default as default44 } from "../icons/aperture";
import { default as default45 } from "../icons/app-window-mac";
import { default as default46 } from "../icons/app-window";
import { default as default47 } from "../icons/apple";
import { default as default48 } from "../icons/archive-restore";
import { default as default49 } from "../icons/archive-x";
import { default as default50 } from "../icons/archive";
import { default as default51 } from "../icons/armchair";
import { default as default52 } from "../icons/arrow-big-down";
import { default as default53 } from "../icons/arrow-big-left-dash";
import { default as default54 } from "../icons/arrow-big-left";
import { default as default55 } from "../icons/arrow-big-right-dash";
import { default as default56 } from "../icons/arrow-big-down-dash";
import { default as default57 } from "../icons/arrow-big-right";
import { default as default58 } from "../icons/arrow-big-up-dash";
import { default as default59 } from "../icons/arrow-big-up";
import { default as default60 } from "../icons/arrow-down-from-line";
import { default as default61 } from "../icons/arrow-down-narrow-wide";
import { default as default62 } from "../icons/arrow-down-left";
import { default as default63 } from "../icons/arrow-down-right";
import { default as default64 } from "../icons/arrow-down-to-line";
import { default as default65 } from "../icons/arrow-down-to-dot";
import { default as default66 } from "../icons/arrow-down-up";
import { default as default67 } from "../icons/arrow-down";
import { default as default68 } from "../icons/arrow-left-from-line";
import { default as default69 } from "../icons/arrow-left-right";
import { default as default70 } from "../icons/arrow-left-to-line";
import { default as default71 } from "../icons/arrow-left";
import { default as default72 } from "../icons/arrow-right-from-line";
import { default as default73 } from "../icons/arrow-right-left";
import { default as default74 } from "../icons/arrow-right-to-line";
import { default as default75 } from "../icons/arrow-right";
import { default as default76 } from "../icons/arrow-up-down";
import { default as default77 } from "../icons/arrow-up-from-dot";
import { default as default78 } from "../icons/arrow-up-from-line";
import { default as default79 } from "../icons/arrow-up-left";
import { default as default80 } from "../icons/arrow-up-to-line";
import { default as default81 } from "../icons/arrow-up-right";
import { default as default82 } from "../icons/arrow-up-wide-narrow";
import { default as default83 } from "../icons/arrow-up";
import { default as default84 } from "../icons/arrows-up-from-line";
import { default as default85 } from "../icons/at-sign";
import { default as default86 } from "../icons/asterisk";
import { default as default87 } from "../icons/atom";
import { default as default88 } from "../icons/audio-lines";
import { default as default89 } from "../icons/award";
import { default as default90 } from "../icons/audio-waveform";
import { default as default91 } from "../icons/axe";
import { default as default92 } from "../icons/baby";
import { default as default93 } from "../icons/backpack";
import { default as default94 } from "../icons/badge-alert";
import { default as default95 } from "../icons/badge-cent";
import { default as default96 } from "../icons/badge-dollar-sign";
import { default as default97 } from "../icons/badge-euro";
import { default as default98 } from "../icons/badge-indian-rupee";
import { default as default99 } from "../icons/badge-info";
import { default as default100 } from "../icons/badge-minus";
import { default as default101 } from "../icons/badge-japanese-yen";
import { default as default102 } from "../icons/badge-percent";
import { default as default103 } from "../icons/badge-plus";
import { default as default104 } from "../icons/badge-pound-sterling";
import { default as default105 } from "../icons/badge-russian-ruble";
import { default as default106 } from "../icons/badge-swiss-franc";
import { default as default107 } from "../icons/badge-x";
import { default as default108 } from "../icons/badge-turkish-lira";
import { default as default109 } from "../icons/baggage-claim";
import { default as default110 } from "../icons/badge";
import { default as default111 } from "../icons/balloon";
import { default as default112 } from "../icons/ban";
import { default as default113 } from "../icons/banana";
import { default as default114 } from "../icons/bandage";
import { default as default115 } from "../icons/banknote-arrow-down";
import { default as default116 } from "../icons/banknote-x";
import { default as default117 } from "../icons/banknote-arrow-up";
import { default as default118 } from "../icons/banknote";
import { default as default119 } from "../icons/barrel";
import { default as default120 } from "../icons/barcode";
import { default as default121 } from "../icons/bath";
import { default as default122 } from "../icons/baseline";
import { default as default123 } from "../icons/battery-charging";
import { default as default124 } from "../icons/battery-full";
import { default as default125 } from "../icons/battery-medium";
import { default as default126 } from "../icons/battery-low";
import { default as default127 } from "../icons/battery-plus";
import { default as default128 } from "../icons/battery-warning";
import { default as default129 } from "../icons/beaker";
import { default as default130 } from "../icons/battery";
import { default as default131 } from "../icons/bean-off";
import { default as default132 } from "../icons/bean";
import { default as default133 } from "../icons/bed-double";
import { default as default134 } from "../icons/bed-single";
import { default as default135 } from "../icons/bed";
import { default as default136 } from "../icons/beef";
import { default as default137 } from "../icons/beer-off";
import { default as default138 } from "../icons/beer";
import { default as default139 } from "../icons/bell-electric";
import { default as default140 } from "../icons/bell-dot";
import { default as default141 } from "../icons/bell-minus";
import { default as default142 } from "../icons/bell-off";
import { default as default143 } from "../icons/bell-plus";
import { default as default144 } from "../icons/bell-ring";
import { default as default145 } from "../icons/bell";
import { default as default146 } from "../icons/between-vertical-end";
import { default as default147 } from "../icons/between-vertical-start";
import { default as default148 } from "../icons/biceps-flexed";
import { default as default149 } from "../icons/bike";
import { default as default150 } from "../icons/binary";
import { default as default151 } from "../icons/biohazard";
import { default as default152 } from "../icons/binoculars";
import { default as default153 } from "../icons/bird";
import { default as default154 } from "../icons/birdhouse";
import { default as default155 } from "../icons/bitcoin";
import { default as default156 } from "../icons/blinds";
import { default as default157 } from "../icons/blend";
import { default as default158 } from "../icons/blocks";
import { default as default159 } from "../icons/bluetooth-connected";
import { default as default160 } from "../icons/bluetooth-off";
import { default as default161 } from "../icons/bluetooth";
import { default as default162 } from "../icons/bluetooth-searching";
import { default as default163 } from "../icons/bold";
import { default as default164 } from "../icons/bolt";
import { default as default165 } from "../icons/bomb";
import { default as default166 } from "../icons/bone";
import { default as default167 } from "../icons/book-a";
import { default as default168 } from "../icons/book-alert";
import { default as default169 } from "../icons/book-audio";
import { default as default170 } from "../icons/book-copy";
import { default as default171 } from "../icons/book-check";
import { default as default172 } from "../icons/book-down";
import { default as default173 } from "../icons/book-headphones";
import { default as default174 } from "../icons/book-heart";
import { default as default175 } from "../icons/book-image";
import { default as default176 } from "../icons/book-key";
import { default as default177 } from "../icons/book-minus";
import { default as default178 } from "../icons/book-marked";
import { default as default179 } from "../icons/book-lock";
import { default as default180 } from "../icons/book-open-check";
import { default as default181 } from "../icons/book-open-text";
import { default as default182 } from "../icons/book-open";
import { default as default183 } from "../icons/book-plus";
import { default as default184 } from "../icons/book-search";
import { default as default185 } from "../icons/book-text";
import { default as default186 } from "../icons/book-type";
import { default as default187 } from "../icons/book-up";
import { default as default188 } from "../icons/book-up-2";
import { default as default189 } from "../icons/book-user";
import { default as default190 } from "../icons/book";
import { default as default191 } from "../icons/bookmark-check";
import { default as default192 } from "../icons/book-x";
import { default as default193 } from "../icons/bookmark-minus";
import { default as default194 } from "../icons/bookmark-plus";
import { default as default195 } from "../icons/bookmark-x";
import { default as default196 } from "../icons/bookmark";
import { default as default197 } from "../icons/boom-box";
import { default as default198 } from "../icons/bot-off";
import { default as default199 } from "../icons/bot-message-square";
import { default as default200 } from "../icons/bot";
import { default as default201 } from "../icons/bottle-wine";
import { default as default202 } from "../icons/bow-arrow";
import { default as default203 } from "../icons/box";
import { default as default204 } from "../icons/boxes";
import { default as default205 } from "../icons/brackets";
import { default as default206 } from "../icons/brain-cog";
import { default as default207 } from "../icons/brain-circuit";
import { default as default208 } from "../icons/brain";
import { default as default209 } from "../icons/brick-wall-shield";
import { default as default210 } from "../icons/brick-wall-fire";
import { default as default211 } from "../icons/brick-wall";
import { default as default212 } from "../icons/briefcase-conveyor-belt";
import { default as default213 } from "../icons/briefcase-business";
import { default as default214 } from "../icons/briefcase-medical";
import { default as default215 } from "../icons/briefcase";
import { default as default216 } from "../icons/bring-to-front";
import { default as default217 } from "../icons/brush-cleaning";
import { default as default218 } from "../icons/brush";
import { default as default219 } from "../icons/bubbles";
import { default as default220 } from "../icons/bug-off";
import { default as default221 } from "../icons/bug-play";
import { default as default222 } from "../icons/bug";
import { default as default223 } from "../icons/building-2";
import { default as default224 } from "../icons/bus-front";
import { default as default225 } from "../icons/building";
import { default as default226 } from "../icons/bus";
import { default as default227 } from "../icons/cable-car";
import { default as default228 } from "../icons/cake-slice";
import { default as default229 } from "../icons/cable";
import { default as default230 } from "../icons/cake";
import { default as default231 } from "../icons/calculator";
import { default as default232 } from "../icons/calendar-1";
import { default as default233 } from "../icons/calendar-arrow-up";
import { default as default234 } from "../icons/calendar-check-2";
import { default as default235 } from "../icons/calendar-arrow-down";
import { default as default236 } from "../icons/calendar-check";
import { default as default237 } from "../icons/calendar-clock";
import { default as default238 } from "../icons/calendar-days";
import { default as default239 } from "../icons/calendar-cog";
import { default as default240 } from "../icons/calendar-fold";
import { default as default241 } from "../icons/calendar-heart";
import { default as default242 } from "../icons/calendar-minus";
import { default as default243 } from "../icons/calendar-minus-2";
import { default as default244 } from "../icons/calendar-off";
import { default as default245 } from "../icons/calendar-plus-2";
import { default as default246 } from "../icons/calendar-plus";
import { default as default247 } from "../icons/calendar-range";
import { default as default248 } from "../icons/calendar-search";
import { default as default249 } from "../icons/calendar-sync";
import { default as default250 } from "../icons/calendar-x-2";
import { default as default251 } from "../icons/calendar";
import { default as default252 } from "../icons/calendar-x";
import { default as default253 } from "../icons/calendars";
import { default as default254 } from "../icons/camera-off";
import { default as default255 } from "../icons/camera";
import { default as default256 } from "../icons/candy-cane";
import { default as default257 } from "../icons/candy";
import { default as default258 } from "../icons/candy-off";
import { default as default259 } from "../icons/cannabis-off";
import { default as default260 } from "../icons/cannabis";
import { default as default261 } from "../icons/captions-off";
import { default as default262 } from "../icons/car-front";
import { default as default263 } from "../icons/car";
import { default as default264 } from "../icons/car-taxi-front";
import { default as default265 } from "../icons/caravan";
import { default as default266 } from "../icons/card-sim";
import { default as default267 } from "../icons/carrot";
import { default as default268 } from "../icons/case-lower";
import { default as default269 } from "../icons/case-sensitive";
import { default as default270 } from "../icons/case-upper";
import { default as default271 } from "../icons/cassette-tape";
import { default as default272 } from "../icons/cast";
import { default as default273 } from "../icons/castle";
import { default as default274 } from "../icons/cat";
import { default as default275 } from "../icons/chart-bar-decreasing";
import { default as default276 } from "../icons/cctv";
import { default as default277 } from "../icons/chart-bar-increasing";
import { default as default278 } from "../icons/chart-bar-stacked";
import { default as default279 } from "../icons/chart-column-stacked";
import { default as default280 } from "../icons/chart-column-decreasing";
import { default as default281 } from "../icons/chart-gantt";
import { default as default282 } from "../icons/chart-network";
import { default as default283 } from "../icons/chart-no-axes-column-decreasing";
import { default as default284 } from "../icons/chart-no-axes-combined";
import { default as default285 } from "../icons/check-check";
import { default as default286 } from "../icons/chart-spline";
import { default as default287 } from "../icons/check-line";
import { default as default288 } from "../icons/check";
import { default as default289 } from "../icons/cherry";
import { default as default290 } from "../icons/chef-hat";
import { default as default291 } from "../icons/chess-king";
import { default as default292 } from "../icons/chess-bishop";
import { default as default293 } from "../icons/chess-knight";
import { default as default294 } from "../icons/chess-pawn";
import { default as default295 } from "../icons/chess-queen";
import { default as default296 } from "../icons/chess-rook";
import { default as default297 } from "../icons/chevron-down";
import { default as default298 } from "../icons/chevron-first";
import { default as default299 } from "../icons/chevron-last";
import { default as default300 } from "../icons/chevron-left";
import { default as default301 } from "../icons/chevron-right";
import { default as default302 } from "../icons/chevron-up";
import { default as default303 } from "../icons/chevrons-down-up";
import { default as default304 } from "../icons/chevrons-down";
import { default as default305 } from "../icons/chevrons-left-right-ellipsis";
import { default as default306 } from "../icons/chevrons-left-right";
import { default as default307 } from "../icons/chevrons-left";
import { default as default308 } from "../icons/chevrons-right-left";
import { default as default309 } from "../icons/chevrons-right";
import { default as default310 } from "../icons/chevrons-up-down";
import { default as default311 } from "../icons/chevrons-up";
import { default as default312 } from "../icons/church";
import { default as default313 } from "../icons/cigarette-off";
import { default as default314 } from "../icons/cigarette";
import { default as default315 } from "../icons/circle-dashed";
import { default as default316 } from "../icons/circle-dollar-sign";
import { default as default317 } from "../icons/circle-dot-dashed";
import { default as default318 } from "../icons/circle-ellipsis";
import { default as default319 } from "../icons/circle-dot";
import { default as default320 } from "../icons/circle-equal";
import { default as default321 } from "../icons/circle-fading-arrow-up";
import { default as default322 } from "../icons/circle-fading-plus";
import { default as default323 } from "../icons/circle-off";
import { default as default324 } from "../icons/circle-pile";
import { default as default325 } from "../icons/circle-pound-sterling";
import { default as default326 } from "../icons/circle-slash";
import { default as default327 } from "../icons/circle-small";
import { default as default328 } from "../icons/circle-star";
import { default as default329 } from "../icons/circle";
import { default as default330 } from "../icons/citrus";
import { default as default331 } from "../icons/circuit-board";
import { default as default332 } from "../icons/clapperboard";
import { default as default333 } from "../icons/clipboard-check";
import { default as default334 } from "../icons/clipboard-clock";
import { default as default335 } from "../icons/clipboard-copy";
import { default as default336 } from "../icons/clipboard-list";
import { default as default337 } from "../icons/clipboard-minus";
import { default as default338 } from "../icons/clipboard-paste";
import { default as default339 } from "../icons/clipboard-plus";
import { default as default340 } from "../icons/clipboard-type";
import { default as default341 } from "../icons/clipboard-x";
import { default as default342 } from "../icons/clipboard";
import { default as default343 } from "../icons/clock-10";
import { default as default344 } from "../icons/clock-1";
import { default as default345 } from "../icons/clock-11";
import { default as default346 } from "../icons/clock-12";
import { default as default347 } from "../icons/clock-2";
import { default as default348 } from "../icons/clock-4";
import { default as default349 } from "../icons/clock-3";
import { default as default350 } from "../icons/clock-5";
import { default as default351 } from "../icons/clock-6";
import { default as default352 } from "../icons/clock-7";
import { default as default353 } from "../icons/clock-8";
import { default as default354 } from "../icons/clock-alert";
import { default as default355 } from "../icons/clock-9";
import { default as default356 } from "../icons/clock-arrow-down";
import { default as default357 } from "../icons/clock-arrow-up";
import { default as default358 } from "../icons/clock-check";
import { default as default359 } from "../icons/clock-fading";
import { default as default360 } from "../icons/clock-plus";
import { default as default361 } from "../icons/clock";
import { default as default362 } from "../icons/closed-caption";
import { default as default363 } from "../icons/cloud-alert";
import { default as default364 } from "../icons/cloud-backup";
import { default as default365 } from "../icons/cloud-cog";
import { default as default366 } from "../icons/cloud-check";
import { default as default367 } from "../icons/cloud-drizzle";
import { default as default368 } from "../icons/cloud-fog";
import { default as default369 } from "../icons/cloud-lightning";
import { default as default370 } from "../icons/cloud-hail";
import { default as default371 } from "../icons/cloud-moon-rain";
import { default as default372 } from "../icons/cloud-off";
import { default as default373 } from "../icons/cloud-moon";
import { default as default374 } from "../icons/cloud-rain";
import { default as default375 } from "../icons/cloud-rain-wind";
import { default as default376 } from "../icons/cloud-snow";
import { default as default377 } from "../icons/cloud-sun-rain";
import { default as default378 } from "../icons/cloud-sun";
import { default as default379 } from "../icons/cloud-sync";
import { default as default380 } from "../icons/cloudy";
import { default as default381 } from "../icons/cloud";
import { default as default382 } from "../icons/clover";
import { default as default383 } from "../icons/club";
import { default as default384 } from "../icons/codepen";
import { default as default385 } from "../icons/code";
import { default as default386 } from "../icons/codesandbox";
import { default as default387 } from "../icons/coffee";
import { default as default388 } from "../icons/cog";
import { default as default389 } from "../icons/coins";
import { default as default390 } from "../icons/columns-4";
import { default as default391 } from "../icons/combine";
import { default as default392 } from "../icons/command";
import { default as default393 } from "../icons/component";
import { default as default394 } from "../icons/compass";
import { default as default395 } from "../icons/computer";
import { default as default396 } from "../icons/concierge-bell";
import { default as default397 } from "../icons/cone";
import { default as default398 } from "../icons/construction";
import { default as default399 } from "../icons/contact";
import { default as default400 } from "../icons/container";
import { default as default401 } from "../icons/contrast";
import { default as default402 } from "../icons/cookie";
import { default as default403 } from "../icons/copy-check";
import { default as default404 } from "../icons/cooking-pot";
import { default as default405 } from "../icons/copy-minus";
import { default as default406 } from "../icons/copy-plus";
import { default as default407 } from "../icons/copy-slash";
import { default as default408 } from "../icons/copy-x";
import { default as default409 } from "../icons/copy";
import { default as default410 } from "../icons/copyleft";
import { default as default411 } from "../icons/copyright";
import { default as default412 } from "../icons/corner-down-left";
import { default as default413 } from "../icons/corner-down-right";
import { default as default414 } from "../icons/corner-left-down";
import { default as default415 } from "../icons/corner-left-up";
import { default as default416 } from "../icons/corner-right-down";
import { default as default417 } from "../icons/corner-right-up";
import { default as default418 } from "../icons/corner-up-left";
import { default as default419 } from "../icons/corner-up-right";
import { default as default420 } from "../icons/cpu";
import { default as default421 } from "../icons/creative-commons";
import { default as default422 } from "../icons/credit-card";
import { default as default423 } from "../icons/crop";
import { default as default424 } from "../icons/croissant";
import { default as default425 } from "../icons/crosshair";
import { default as default426 } from "../icons/cross";
import { default as default427 } from "../icons/cuboid";
import { default as default428 } from "../icons/crown";
import { default as default429 } from "../icons/cup-soda";
import { default as default430 } from "../icons/currency";
import { default as default431 } from "../icons/cylinder";
import { default as default432 } from "../icons/dam";
import { default as default433 } from "../icons/database-backup";
import { default as default434 } from "../icons/database-search";
import { default as default435 } from "../icons/database-zap";
import { default as default436 } from "../icons/decimals-arrow-left";
import { default as default437 } from "../icons/database";
import { default as default438 } from "../icons/decimals-arrow-right";
import { default as default439 } from "../icons/delete";
import { default as default440 } from "../icons/diameter";
import { default as default441 } from "../icons/dessert";
import { default as default442 } from "../icons/diamond-minus";
import { default as default443 } from "../icons/diamond";
import { default as default444 } from "../icons/diamond-plus";
import { default as default445 } from "../icons/dice-1";
import { default as default446 } from "../icons/dice-2";
import { default as default447 } from "../icons/dice-3";
import { default as default448 } from "../icons/dice-4";
import { default as default449 } from "../icons/dice-5";
import { default as default450 } from "../icons/dices";
import { default as default451 } from "../icons/dice-6";
import { default as default452 } from "../icons/diff";
import { default as default453 } from "../icons/disc-2";
import { default as default454 } from "../icons/disc-3";
import { default as default455 } from "../icons/disc-album";
import { default as default456 } from "../icons/divide";
import { default as default457 } from "../icons/disc";
import { default as default458 } from "../icons/dna-off";
import { default as default459 } from "../icons/dna";
import { default as default460 } from "../icons/dock";
import { default as default461 } from "../icons/dog";
import { default as default462 } from "../icons/donut";
import { default as default463 } from "../icons/dollar-sign";
import { default as default464 } from "../icons/door-closed-locked";
import { default as default465 } from "../icons/door-open";
import { default as default466 } from "../icons/door-closed";
import { default as default467 } from "../icons/dot";
import { default as default468 } from "../icons/download";
import { default as default469 } from "../icons/drafting-compass";
import { default as default470 } from "../icons/drama";
import { default as default471 } from "../icons/dribbble";
import { default as default472 } from "../icons/drill";
import { default as default473 } from "../icons/drone";
import { default as default474 } from "../icons/droplet";
import { default as default475 } from "../icons/droplets";
import { default as default476 } from "../icons/drum";
import { default as default477 } from "../icons/droplet-off";
import { default as default478 } from "../icons/drumstick";
import { default as default479 } from "../icons/dumbbell";
import { default as default480 } from "../icons/ear";
import { default as default481 } from "../icons/earth-lock";
import { default as default482 } from "../icons/ear-off";
import { default as default483 } from "../icons/eclipse";
import { default as default484 } from "../icons/egg-fried";
import { default as default485 } from "../icons/egg-off";
import { default as default486 } from "../icons/egg";
import { default as default487 } from "../icons/ellipse";
import { default as default488 } from "../icons/equal-approximately";
import { default as default489 } from "../icons/equal-not";
import { default as default490 } from "../icons/equal";
import { default as default491 } from "../icons/eraser";
import { default as default492 } from "../icons/ethernet-port";
import { default as default493 } from "../icons/euro";
import { default as default494 } from "../icons/ev-charger";
import { default as default495 } from "../icons/expand";
import { default as default496 } from "../icons/external-link";
import { default as default497 } from "../icons/eye-closed";
import { default as default498 } from "../icons/eye-off";
import { default as default499 } from "../icons/eye";
import { default as default500 } from "../icons/facebook";
import { default as default501 } from "../icons/factory";
import { default as default502 } from "../icons/fan";
import { default as default503 } from "../icons/fast-forward";
import { default as default504 } from "../icons/feather";
import { default as default505 } from "../icons/fence";
import { default as default506 } from "../icons/ferris-wheel";
import { default as default507 } from "../icons/figma";
import { default as default508 } from "../icons/file-archive";
import { default as default509 } from "../icons/file-box";
import { default as default510 } from "../icons/file-check";
import { default as default511 } from "../icons/file-clock";
import { default as default512 } from "../icons/file-code";
import { default as default513 } from "../icons/file-diff";
import { default as default514 } from "../icons/file-digit";
import { default as default515 } from "../icons/file-down";
import { default as default516 } from "../icons/file-image";
import { default as default517 } from "../icons/file-heart";
import { default as default518 } from "../icons/file-input";
import { default as default519 } from "../icons/file-minus";
import { default as default520 } from "../icons/file-music";
import { default as default521 } from "../icons/file-output";
import { default as default522 } from "../icons/file-plus";
import { default as default523 } from "../icons/file-scan";
import { default as default524 } from "../icons/file-search";
import { default as default525 } from "../icons/file-sliders";
import { default as default526 } from "../icons/file-spreadsheet";
import { default as default527 } from "../icons/file-stack";
import { default as default528 } from "../icons/file-symlink";
import { default as default529 } from "../icons/file-terminal";
import { default as default530 } from "../icons/file-type";
import { default as default531 } from "../icons/file-text";
import { default as default532 } from "../icons/file-user";
import { default as default533 } from "../icons/file-up";
import { default as default534 } from "../icons/file-volume";
import { default as default535 } from "../icons/file-x";
import { default as default536 } from "../icons/file";
import { default as default537 } from "../icons/files";
import { default as default538 } from "../icons/film";
import { default as default539 } from "../icons/fire-extinguisher";
import { default as default540 } from "../icons/fish-off";
import { default as default541 } from "../icons/fish";
import { default as default542 } from "../icons/fish-symbol";
import { default as default543 } from "../icons/fishing-hook";
import { default as default544 } from "../icons/fishing-rod";
import { default as default545 } from "../icons/flag-off";
import { default as default546 } from "../icons/flag-triangle-left";
import { default as default547 } from "../icons/flag-triangle-right";
import { default as default548 } from "../icons/flag";
import { default as default549 } from "../icons/flame-kindling";
import { default as default550 } from "../icons/flashlight-off";
import { default as default551 } from "../icons/flame";
import { default as default552 } from "../icons/flask-conical-off";
import { default as default553 } from "../icons/flashlight";
import { default as default554 } from "../icons/flask-conical";
import { default as default555 } from "../icons/flask-round";
import { default as default556 } from "../icons/flip-horizontal-2";
import { default as default557 } from "../icons/flip-vertical-2";
import { default as default558 } from "../icons/flower-2";
import { default as default559 } from "../icons/flower";
import { default as default560 } from "../icons/fold-horizontal";
import { default as default561 } from "../icons/focus";
import { default as default562 } from "../icons/fold-vertical";
import { default as default563 } from "../icons/folder-archive";
import { default as default564 } from "../icons/folder-check";
import { default as default565 } from "../icons/folder-clock";
import { default as default566 } from "../icons/folder-closed";
import { default as default567 } from "../icons/folder-code";
import { default as default568 } from "../icons/folder-dot";
import { default as default569 } from "../icons/folder-down";
import { default as default570 } from "../icons/folder-git-2";
import { default as default571 } from "../icons/folder-git";
import { default as default572 } from "../icons/folder-heart";
import { default as default573 } from "../icons/folder-input";
import { default as default574 } from "../icons/folder-key";
import { default as default575 } from "../icons/folder-kanban";
import { default as default576 } from "../icons/folder-lock";
import { default as default577 } from "../icons/folder-minus";
import { default as default578 } from "../icons/folder-open-dot";
import { default as default579 } from "../icons/folder-open";
import { default as default580 } from "../icons/folder-output";
import { default as default581 } from "../icons/folder-plus";
import { default as default582 } from "../icons/folder-root";
import { default as default583 } from "../icons/folder-search-2";
import { default as default584 } from "../icons/folder-search";
import { default as default585 } from "../icons/folder-sync";
import { default as default586 } from "../icons/folder-symlink";
import { default as default587 } from "../icons/folder-tree";
import { default as default588 } from "../icons/folder-up";
import { default as default589 } from "../icons/folder-x";
import { default as default590 } from "../icons/folders";
import { default as default591 } from "../icons/footprints";
import { default as default592 } from "../icons/folder";
import { default as default593 } from "../icons/forklift";
import { default as default594 } from "../icons/form";
import { default as default595 } from "../icons/forward";
import { default as default596 } from "../icons/frame";
import { default as default597 } from "../icons/frown";
import { default as default598 } from "../icons/framer";
import { default as default599 } from "../icons/fuel";
import { default as default600 } from "../icons/fullscreen";
import { default as default601 } from "../icons/funnel-plus";
import { default as default602 } from "../icons/gallery-horizontal";
import { default as default603 } from "../icons/gallery-thumbnails";
import { default as default604 } from "../icons/gallery-horizontal-end";
import { default as default605 } from "../icons/gallery-vertical";
import { default as default606 } from "../icons/gallery-vertical-end";
import { default as default607 } from "../icons/gamepad-2";
import { default as default608 } from "../icons/gamepad-directional";
import { default as default609 } from "../icons/gamepad";
import { default as default610 } from "../icons/gauge";
import { default as default611 } from "../icons/gavel";
import { default as default612 } from "../icons/gem";
import { default as default613 } from "../icons/georgian-lari";
import { default as default614 } from "../icons/ghost";
import { default as default615 } from "../icons/git-branch-minus";
import { default as default616 } from "../icons/git-branch";
import { default as default617 } from "../icons/gift";
import { default as default618 } from "../icons/git-branch-plus";
import { default as default619 } from "../icons/git-commit-vertical";
import { default as default620 } from "../icons/git-compare-arrows";
import { default as default621 } from "../icons/git-compare";
import { default as default622 } from "../icons/git-fork";
import { default as default623 } from "../icons/git-graph";
import { default as default624 } from "../icons/git-merge-conflict";
import { default as default625 } from "../icons/git-merge";
import { default as default626 } from "../icons/git-pull-request-arrow";
import { default as default627 } from "../icons/git-pull-request-closed";
import { default as default628 } from "../icons/git-pull-request-create-arrow";
import { default as default629 } from "../icons/git-pull-request-draft";
import { default as default630 } from "../icons/git-pull-request-create";
import { default as default631 } from "../icons/git-pull-request";
import { default as default632 } from "../icons/github";
import { default as default633 } from "../icons/gitlab";
import { default as default634 } from "../icons/glass-water";
import { default as default635 } from "../icons/glasses";
import { default as default636 } from "../icons/globe-lock";
import { default as default637 } from "../icons/globe-x";
import { default as default638 } from "../icons/globe-off";
import { default as default639 } from "../icons/goal";
import { default as default640 } from "../icons/gpu";
import { default as default641 } from "../icons/globe";
import { default as default642 } from "../icons/graduation-cap";
import { default as default643 } from "../icons/grape";
import { default as default644 } from "../icons/grid-3x2";
import { default as default645 } from "../icons/grip-horizontal";
import { default as default646 } from "../icons/grip-vertical";
import { default as default647 } from "../icons/grip";
import { default as default648 } from "../icons/group";
import { default as default649 } from "../icons/ham";
import { default as default650 } from "../icons/hamburger";
import { default as default651 } from "../icons/hammer";
import { default as default652 } from "../icons/guitar";
import { default as default653 } from "../icons/hand-coins";
import { default as default654 } from "../icons/hand-fist";
import { default as default655 } from "../icons/hand-heart";
import { default as default656 } from "../icons/hand-metal";
import { default as default657 } from "../icons/hand";
import { default as default658 } from "../icons/hand-platter";
import { default as default659 } from "../icons/handbag";
import { default as default660 } from "../icons/handshake";
import { default as default661 } from "../icons/hard-drive-upload";
import { default as default662 } from "../icons/hard-drive-download";
import { default as default663 } from "../icons/hard-drive";
import { default as default664 } from "../icons/hard-hat";
import { default as default665 } from "../icons/hash";
import { default as default666 } from "../icons/hat-glasses";
import { default as default667 } from "../icons/haze";
import { default as default668 } from "../icons/hd";
import { default as default669 } from "../icons/hdmi-port";
import { default as default670 } from "../icons/heading-1";
import { default as default671 } from "../icons/heading-2";
import { default as default672 } from "../icons/heading-3";
import { default as default673 } from "../icons/heading-4";
import { default as default674 } from "../icons/heading-6";
import { default as default675 } from "../icons/heading-5";
import { default as default676 } from "../icons/heading";
import { default as default677 } from "../icons/headphone-off";
import { default as default678 } from "../icons/headphones";
import { default as default679 } from "../icons/headset";
import { default as default680 } from "../icons/heart-crack";
import { default as default681 } from "../icons/heart-handshake";
import { default as default682 } from "../icons/heart-minus";
import { default as default683 } from "../icons/heart-off";
import { default as default684 } from "../icons/heart-plus";
import { default as default685 } from "../icons/heart-pulse";
import { default as default686 } from "../icons/heart";
import { default as default687 } from "../icons/heater";
import { default as default688 } from "../icons/helicopter";
import { default as default689 } from "../icons/hexagon";
import { default as default690 } from "../icons/highlighter";
import { default as default691 } from "../icons/history";
import { default as default692 } from "../icons/hop-off";
import { default as default693 } from "../icons/hop";
import { default as default694 } from "../icons/hotel";
import { default as default695 } from "../icons/hospital";
import { default as default696 } from "../icons/hourglass";
import { default as default697 } from "../icons/house-heart";
import { default as default698 } from "../icons/house-plug";
import { default as default699 } from "../icons/house-wifi";
import { default as default700 } from "../icons/house-plus";
import { default as default701 } from "../icons/id-card-lanyard";
import { default as default702 } from "../icons/image-down";
import { default as default703 } from "../icons/id-card";
import { default as default704 } from "../icons/image-minus";
import { default as default705 } from "../icons/image-play";
import { default as default706 } from "../icons/image-off";
import { default as default707 } from "../icons/image-plus";
import { default as default708 } from "../icons/image-up";
import { default as default709 } from "../icons/image-upscale";
import { default as default710 } from "../icons/image";
import { default as default711 } from "../icons/import";
import { default as default712 } from "../icons/images";
import { default as default713 } from "../icons/indian-rupee";
import { default as default714 } from "../icons/inbox";
import { default as default715 } from "../icons/infinity";
import { default as default716 } from "../icons/info";
import { default as default717 } from "../icons/inspection-panel";
import { default as default718 } from "../icons/instagram";
import { default as default719 } from "../icons/italic";
import { default as default720 } from "../icons/iteration-ccw";
import { default as default721 } from "../icons/iteration-cw";
import { default as default722 } from "../icons/japanese-yen";
import { default as default723 } from "../icons/joystick";
import { default as default724 } from "../icons/kanban";
import { default as default725 } from "../icons/kayak";
import { default as default726 } from "../icons/key-round";
import { default as default727 } from "../icons/key-square";
import { default as default728 } from "../icons/key";
import { default as default729 } from "../icons/keyboard-music";
import { default as default730 } from "../icons/keyboard";
import { default as default731 } from "../icons/keyboard-off";
import { default as default732 } from "../icons/lamp-desk";
import { default as default733 } from "../icons/lamp-floor";
import { default as default734 } from "../icons/lamp-ceiling";
import { default as default735 } from "../icons/lamp-wall-down";
import { default as default736 } from "../icons/lamp-wall-up";
import { default as default737 } from "../icons/lamp";
import { default as default738 } from "../icons/land-plot";
import { default as default739 } from "../icons/landmark";
import { default as default740 } from "../icons/languages";
import { default as default741 } from "../icons/laptop-minimal-check";
import { default as default742 } from "../icons/laptop";
import { default as default743 } from "../icons/lasso-select";
import { default as default744 } from "../icons/laugh";
import { default as default745 } from "../icons/lasso";
import { default as default746 } from "../icons/layers-2";
import { default as default747 } from "../icons/layers-plus";
import { default as default748 } from "../icons/layout-dashboard";
import { default as default749 } from "../icons/layout-list";
import { default as default750 } from "../icons/layout-grid";
import { default as default751 } from "../icons/layout-panel-left";
import { default as default752 } from "../icons/layout-template";
import { default as default753 } from "../icons/layout-panel-top";
import { default as default754 } from "../icons/lectern";
import { default as default755 } from "../icons/leaf";
import { default as default756 } from "../icons/lens-concave";
import { default as default757 } from "../icons/lens-convex";
import { default as default758 } from "../icons/leafy-green";
import { default as default759 } from "../icons/library-big";
import { default as default760 } from "../icons/library";
import { default as default761 } from "../icons/life-buoy";
import { default as default762 } from "../icons/ligature";
import { default as default763 } from "../icons/lightbulb-off";
import { default as default764 } from "../icons/lightbulb";
import { default as default765 } from "../icons/line-dot-right-horizontal";
import { default as default766 } from "../icons/line-squiggle";
import { default as default767 } from "../icons/link-2-off";
import { default as default768 } from "../icons/link-2";
import { default as default769 } from "../icons/link";
import { default as default770 } from "../icons/list-check";
import { default as default771 } from "../icons/linkedin";
import { default as default772 } from "../icons/list-checks";
import { default as default773 } from "../icons/list-chevrons-down-up";
import { default as default774 } from "../icons/list-chevrons-up-down";
import { default as default775 } from "../icons/list-collapse";
import { default as default776 } from "../icons/list-end";
import { default as default777 } from "../icons/list-filter-plus";
import { default as default778 } from "../icons/list-filter";
import { default as default779 } from "../icons/list-minus";
import { default as default780 } from "../icons/list-music";
import { default as default781 } from "../icons/list-ordered";
import { default as default782 } from "../icons/list-plus";
import { default as default783 } from "../icons/list-restart";
import { default as default784 } from "../icons/list-start";
import { default as default785 } from "../icons/list-todo";
import { default as default786 } from "../icons/list-tree";
import { default as default787 } from "../icons/list-video";
import { default as default788 } from "../icons/list-x";
import { default as default789 } from "../icons/loader-pinwheel";
import { default as default790 } from "../icons/list";
import { default as default791 } from "../icons/loader";
import { default as default792 } from "../icons/locate-fixed";
import { default as default793 } from "../icons/locate-off";
import { default as default794 } from "../icons/locate";
import { default as default795 } from "../icons/lock-keyhole";
import { default as default796 } from "../icons/lock";
import { default as default797 } from "../icons/log-in";
import { default as default798 } from "../icons/log-out";
import { default as default799 } from "../icons/logs";
import { default as default800 } from "../icons/lollipop";
import { default as default801 } from "../icons/luggage";
import { default as default802 } from "../icons/mail-check";
import { default as default803 } from "../icons/magnet";
import { default as default804 } from "../icons/mail-minus";
import { default as default805 } from "../icons/mail-open";
import { default as default806 } from "../icons/mail-search";
import { default as default807 } from "../icons/mail-plus";
import { default as default808 } from "../icons/mail-warning";
import { default as default809 } from "../icons/mail-x";
import { default as default810 } from "../icons/mail";
import { default as default811 } from "../icons/mailbox";
import { default as default812 } from "../icons/mails";
import { default as default813 } from "../icons/map-pin-check-inside";
import { default as default814 } from "../icons/map-minus";
import { default as default815 } from "../icons/map-pin-check";
import { default as default816 } from "../icons/map-pin-house";
import { default as default817 } from "../icons/map-pin-minus-inside";
import { default as default818 } from "../icons/map-pin-minus";
import { default as default819 } from "../icons/map-pin-off";
import { default as default820 } from "../icons/map-pin-plus";
import { default as default821 } from "../icons/map-pin-plus-inside";
import { default as default822 } from "../icons/map-pin-x-inside";
import { default as default823 } from "../icons/map-pin-x";
import { default as default824 } from "../icons/map-pin";
import { default as default825 } from "../icons/map-pinned";
import { default as default826 } from "../icons/map-plus";
import { default as default827 } from "../icons/map";
import { default as default828 } from "../icons/mars-stroke";
import { default as default829 } from "../icons/mars";
import { default as default830 } from "../icons/martini";
import { default as default831 } from "../icons/maximize-2";
import { default as default832 } from "../icons/maximize";
import { default as default833 } from "../icons/medal";
import { default as default834 } from "../icons/megaphone-off";
import { default as default835 } from "../icons/megaphone";
import { default as default836 } from "../icons/meh";
import { default as default837 } from "../icons/memory-stick";
import { default as default838 } from "../icons/menu";
import { default as default839 } from "../icons/merge";
import { default as default840 } from "../icons/message-circle-check";
import { default as default841 } from "../icons/message-circle-code";
import { default as default842 } from "../icons/message-circle-dashed";
import { default as default843 } from "../icons/message-circle-heart";
import { default as default844 } from "../icons/message-circle-more";
import { default as default845 } from "../icons/message-circle-plus";
import { default as default846 } from "../icons/message-circle-off";
import { default as default847 } from "../icons/message-circle-reply";
import { default as default848 } from "../icons/message-circle-warning";
import { default as default849 } from "../icons/message-circle-x";
import { default as default850 } from "../icons/message-circle";
import { default as default851 } from "../icons/message-square-check";
import { default as default852 } from "../icons/message-square-dashed";
import { default as default853 } from "../icons/message-square-diff";
import { default as default854 } from "../icons/message-square-code";
import { default as default855 } from "../icons/message-square-dot";
import { default as default856 } from "../icons/message-square-heart";
import { default as default857 } from "../icons/message-square-more";
import { default as default858 } from "../icons/message-square-lock";
import { default as default859 } from "../icons/message-square-off";
import { default as default860 } from "../icons/message-square-plus";
import { default as default861 } from "../icons/message-square-quote";
import { default as default862 } from "../icons/message-square-reply";
import { default as default863 } from "../icons/message-square-share";
import { default as default864 } from "../icons/message-square-text";
import { default as default865 } from "../icons/message-square-warning";
import { default as default866 } from "../icons/message-square";
import { default as default867 } from "../icons/message-square-x";
import { default as default868 } from "../icons/messages-square";
import { default as default869 } from "../icons/metronome";
import { default as default870 } from "../icons/mic-off";
import { default as default871 } from "../icons/microchip";
import { default as default872 } from "../icons/mic";
import { default as default873 } from "../icons/microwave";
import { default as default874 } from "../icons/microscope";
import { default as default875 } from "../icons/milestone";
import { default as default876 } from "../icons/milk-off";
import { default as default877 } from "../icons/minimize-2";
import { default as default878 } from "../icons/milk";
import { default as default879 } from "../icons/minus";
import { default as default880 } from "../icons/minimize";
import { default as default881 } from "../icons/mirror-rectangular";
import { default as default882 } from "../icons/mirror-round";
import { default as default883 } from "../icons/monitor-check";
import { default as default884 } from "../icons/monitor-cloud";
import { default as default885 } from "../icons/monitor-cog";
import { default as default886 } from "../icons/monitor-dot";
import { default as default887 } from "../icons/monitor-off";
import { default as default888 } from "../icons/monitor-down";
import { default as default889 } from "../icons/monitor-play";
import { default as default890 } from "../icons/monitor-smartphone";
import { default as default891 } from "../icons/monitor-speaker";
import { default as default892 } from "../icons/monitor-pause";
import { default as default893 } from "../icons/monitor-stop";
import { default as default894 } from "../icons/monitor-up";
import { default as default895 } from "../icons/monitor-x";
import { default as default896 } from "../icons/monitor";
import { default as default897 } from "../icons/moon-star";
import { default as default898 } from "../icons/moon";
import { default as default899 } from "../icons/motorbike";
import { default as default900 } from "../icons/mountain-snow";
import { default as default901 } from "../icons/mountain";
import { default as default902 } from "../icons/mouse-left";
import { default as default903 } from "../icons/mouse-off";
import { default as default904 } from "../icons/mouse-pointer-2-off";
import { default as default905 } from "../icons/mouse-pointer-2";
import { default as default906 } from "../icons/mouse-pointer-ban";
import { default as default907 } from "../icons/mouse-pointer-click";
import { default as default908 } from "../icons/mouse-right";
import { default as default909 } from "../icons/mouse-pointer";
import { default as default910 } from "../icons/move-diagonal-2";
import { default as default911 } from "../icons/mouse";
import { default as default912 } from "../icons/move-diagonal";
import { default as default913 } from "../icons/move-down-left";
import { default as default914 } from "../icons/move-down-right";
import { default as default915 } from "../icons/move-down";
import { default as default916 } from "../icons/move-left";
import { default as default917 } from "../icons/move-horizontal";
import { default as default918 } from "../icons/move-up-left";
import { default as default919 } from "../icons/move-right";
import { default as default920 } from "../icons/move-up-right";
import { default as default921 } from "../icons/move-up";
import { default as default922 } from "../icons/move-vertical";
import { default as default923 } from "../icons/move";
import { default as default924 } from "../icons/music-2";
import { default as default925 } from "../icons/music-3";
import { default as default926 } from "../icons/music-4";
import { default as default927 } from "../icons/music";
import { default as default928 } from "../icons/navigation-2-off";
import { default as default929 } from "../icons/navigation-2";
import { default as default930 } from "../icons/navigation-off";
import { default as default931 } from "../icons/navigation";
import { default as default932 } from "../icons/network";
import { default as default933 } from "../icons/newspaper";
import { default as default934 } from "../icons/nfc";
import { default as default935 } from "../icons/non-binary";
import { default as default936 } from "../icons/notebook-pen";
import { default as default937 } from "../icons/notebook-tabs";
import { default as default938 } from "../icons/notebook-text";
import { default as default939 } from "../icons/notebook";
import { default as default940 } from "../icons/notepad-text-dashed";
import { default as default941 } from "../icons/notepad-text";
import { default as default942 } from "../icons/nut-off";
import { default as default943 } from "../icons/nut";
import { default as default944 } from "../icons/octagon-minus";
import { default as default945 } from "../icons/octagon";
import { default as default946 } from "../icons/omega";
import { default as default947 } from "../icons/option";
import { default as default948 } from "../icons/origami";
import { default as default949 } from "../icons/orbit";
import { default as default950 } from "../icons/package-2";
import { default as default951 } from "../icons/package-check";
import { default as default952 } from "../icons/package-open";
import { default as default953 } from "../icons/package-minus";
import { default as default954 } from "../icons/package-plus";
import { default as default955 } from "../icons/package-search";
import { default as default956 } from "../icons/package-x";
import { default as default957 } from "../icons/package";
import { default as default958 } from "../icons/paint-bucket";
import { default as default959 } from "../icons/paint-roller";
import { default as default960 } from "../icons/paintbrush";
import { default as default961 } from "../icons/palette";
import { default as default962 } from "../icons/panda";
import { default as default963 } from "../icons/panel-bottom-close";
import { default as default964 } from "../icons/panel-bottom";
import { default as default965 } from "../icons/panel-bottom-open";
import { default as default966 } from "../icons/panel-left-right-dashed";
import { default as default967 } from "../icons/panel-right-close";
import { default as default968 } from "../icons/panel-right-open";
import { default as default969 } from "../icons/panel-right";
import { default as default970 } from "../icons/panel-top-bottom-dashed";
import { default as default971 } from "../icons/panel-top-close";
import { default as default972 } from "../icons/panel-top-open";
import { default as default973 } from "../icons/panel-top";
import { default as default974 } from "../icons/panels-left-bottom";
import { default as default975 } from "../icons/panels-right-bottom";
import { default as default976 } from "../icons/paperclip";
import { default as default977 } from "../icons/parentheses";
import { default as default978 } from "../icons/parking-meter";
import { default as default979 } from "../icons/party-popper";
import { default as default980 } from "../icons/paw-print";
import { default as default981 } from "../icons/pause";
import { default as default982 } from "../icons/pc-case";
import { default as default983 } from "../icons/pen-off";
import { default as default984 } from "../icons/pen-tool";
import { default as default985 } from "../icons/pencil-off";
import { default as default986 } from "../icons/pencil-line";
import { default as default987 } from "../icons/pencil-ruler";
import { default as default988 } from "../icons/pencil";
import { default as default989 } from "../icons/pentagon";
import { default as default990 } from "../icons/person-standing";
import { default as default991 } from "../icons/percent";
import { default as default992 } from "../icons/philippine-peso";
import { default as default993 } from "../icons/phone-call";
import { default as default994 } from "../icons/phone-forwarded";
import { default as default995 } from "../icons/phone-incoming";
import { default as default996 } from "../icons/phone-missed";
import { default as default997 } from "../icons/phone-outgoing";
import { default as default998 } from "../icons/phone-off";
import { default as default999 } from "../icons/phone";
import { default as default1000 } from "../icons/pi";
import { default as default1001 } from "../icons/piano";
import { default as default1002 } from "../icons/pickaxe";
import { default as default1003 } from "../icons/picture-in-picture";
import { default as default1004 } from "../icons/picture-in-picture-2";
import { default as default1005 } from "../icons/piggy-bank";
import { default as default1006 } from "../icons/pilcrow-left";
import { default as default1007 } from "../icons/pilcrow-right";
import { default as default1008 } from "../icons/pilcrow";
import { default as default1009 } from "../icons/pill";
import { default as default1010 } from "../icons/pill-bottle";
import { default as default1011 } from "../icons/pin-off";
import { default as default1012 } from "../icons/pin";
import { default as default1013 } from "../icons/pizza";
import { default as default1014 } from "../icons/pipette";
import { default as default1015 } from "../icons/plane-landing";
import { default as default1016 } from "../icons/plane-takeoff";
import { default as default1017 } from "../icons/plane";
import { default as default1018 } from "../icons/play";
import { default as default1019 } from "../icons/plug-2";
import { default as default1020 } from "../icons/plug";
import { default as default1021 } from "../icons/plus";
import { default as default1022 } from "../icons/pocket-knife";
import { default as default1023 } from "../icons/pocket";
import { default as default1024 } from "../icons/podcast";
import { default as default1025 } from "../icons/pointer-off";
import { default as default1026 } from "../icons/popcorn";
import { default as default1027 } from "../icons/pointer";
import { default as default1028 } from "../icons/popsicle";
import { default as default1029 } from "../icons/pound-sterling";
import { default as default1030 } from "../icons/power-off";
import { default as default1031 } from "../icons/power";
import { default as default1032 } from "../icons/printer-check";
import { default as default1033 } from "../icons/printer-x";
import { default as default1034 } from "../icons/presentation";
import { default as default1035 } from "../icons/printer";
import { default as default1036 } from "../icons/projector";
import { default as default1037 } from "../icons/proportions";
import { default as default1038 } from "../icons/puzzle";
import { default as default1039 } from "../icons/pyramid";
import { default as default1040 } from "../icons/quote";
import { default as default1041 } from "../icons/radar";
import { default as default1042 } from "../icons/qr-code";
import { default as default1043 } from "../icons/rabbit";
import { default as default1044 } from "../icons/radiation";
import { default as default1045 } from "../icons/radical";
import { default as default1046 } from "../icons/radio-receiver";
import { default as default1047 } from "../icons/radio-tower";
import { default as default1048 } from "../icons/radio";
import { default as default1049 } from "../icons/radius";
import { default as default1050 } from "../icons/rainbow";
import { default as default1051 } from "../icons/rail-symbol";
import { default as default1052 } from "../icons/rat";
import { default as default1053 } from "../icons/receipt-cent";
import { default as default1054 } from "../icons/ratio";
import { default as default1055 } from "../icons/receipt-euro";
import { default as default1056 } from "../icons/receipt-indian-rupee";
import { default as default1057 } from "../icons/receipt-pound-sterling";
import { default as default1058 } from "../icons/receipt-japanese-yen";
import { default as default1059 } from "../icons/receipt-russian-ruble";
import { default as default1060 } from "../icons/receipt-turkish-lira";
import { default as default1061 } from "../icons/receipt-text";
import { default as default1062 } from "../icons/receipt";
import { default as default1063 } from "../icons/receipt-swiss-franc";
import { default as default1064 } from "../icons/rectangle-circle";
import { default as default1065 } from "../icons/rectangle-goggles";
import { default as default1066 } from "../icons/recycle";
import { default as default1067 } from "../icons/rectangle-horizontal";
import { default as default1068 } from "../icons/rectangle-vertical";
import { default as default1069 } from "../icons/redo-2";
import { default as default1070 } from "../icons/redo-dot";
import { default as default1071 } from "../icons/refresh-ccw-dot";
import { default as default1072 } from "../icons/redo";
import { default as default1073 } from "../icons/refresh-cw-off";
import { default as default1074 } from "../icons/refresh-ccw";
import { default as default1075 } from "../icons/refresh-cw";
import { default as default1076 } from "../icons/regex";
import { default as default1077 } from "../icons/remove-formatting";
import { default as default1078 } from "../icons/refrigerator";
import { default as default1079 } from "../icons/repeat-1";
import { default as default1080 } from "../icons/repeat-2";
import { default as default1081 } from "../icons/repeat";
import { default as default1082 } from "../icons/replace-all";
import { default as default1083 } from "../icons/replace";
import { default as default1084 } from "../icons/reply";
import { default as default1085 } from "../icons/rewind";
import { default as default1086 } from "../icons/reply-all";
import { default as default1087 } from "../icons/ribbon";
import { default as default1088 } from "../icons/rocket";
import { default as default1089 } from "../icons/roller-coaster";
import { default as default1090 } from "../icons/rocking-chair";
import { default as default1091 } from "../icons/rose";
import { default as default1092 } from "../icons/rotate-ccw-key";
import { default as default1093 } from "../icons/rotate-ccw-square";
import { default as default1094 } from "../icons/rotate-cw-square";
import { default as default1095 } from "../icons/rotate-ccw";
import { default as default1096 } from "../icons/rotate-cw";
import { default as default1097 } from "../icons/route-off";
import { default as default1098 } from "../icons/route";
import { default as default1099 } from "../icons/router";
import { default as default1100 } from "../icons/rss";
import { default as default1101 } from "../icons/rows-4";
import { default as default1102 } from "../icons/ruler-dimension-line";
import { default as default1103 } from "../icons/ruler";
import { default as default1104 } from "../icons/russian-ruble";
import { default as default1105 } from "../icons/sailboat";
import { default as default1106 } from "../icons/salad";
import { default as default1107 } from "../icons/sandwich";
import { default as default1108 } from "../icons/satellite-dish";
import { default as default1109 } from "../icons/satellite";
import { default as default1110 } from "../icons/saudi-riyal";
import { default as default1111 } from "../icons/save-all";
import { default as default1112 } from "../icons/save";
import { default as default1113 } from "../icons/save-off";
import { default as default1114 } from "../icons/scale";
import { default as default1115 } from "../icons/scaling";
import { default as default1116 } from "../icons/scan-barcode";
import { default as default1117 } from "../icons/scan-eye";
import { default as default1118 } from "../icons/scan-face";
import { default as default1119 } from "../icons/scan-heart";
import { default as default1120 } from "../icons/scan-line";
import { default as default1121 } from "../icons/scan-qr-code";
import { default as default1122 } from "../icons/scan-search";
import { default as default1123 } from "../icons/scan-text";
import { default as default1124 } from "../icons/scan";
import { default as default1125 } from "../icons/school";
import { default as default1126 } from "../icons/scissors";
import { default as default1127 } from "../icons/scissors-line-dashed";
import { default as default1128 } from "../icons/scooter";
import { default as default1129 } from "../icons/screen-share";
import { default as default1130 } from "../icons/screen-share-off";
import { default as default1131 } from "../icons/scroll-text";
import { default as default1132 } from "../icons/scroll";
import { default as default1133 } from "../icons/search-alert";
import { default as default1134 } from "../icons/search-check";
import { default as default1135 } from "../icons/search-code";
import { default as default1136 } from "../icons/search-slash";
import { default as default1137 } from "../icons/search";
import { default as default1138 } from "../icons/search-x";
import { default as default1139 } from "../icons/section";
import { default as default1140 } from "../icons/send-to-back";
import { default as default1141 } from "../icons/send";
import { default as default1142 } from "../icons/separator-horizontal";
import { default as default1143 } from "../icons/separator-vertical";
import { default as default1144 } from "../icons/server-cog";
import { default as default1145 } from "../icons/server-crash";
import { default as default1146 } from "../icons/server-off";
import { default as default1147 } from "../icons/server";
import { default as default1148 } from "../icons/settings-2";
import { default as default1149 } from "../icons/shapes";
import { default as default1150 } from "../icons/settings";
import { default as default1151 } from "../icons/share-2";
import { default as default1152 } from "../icons/share";
import { default as default1153 } from "../icons/sheet";
import { default as default1154 } from "../icons/shell";
import { default as default1155 } from "../icons/shelving-unit";
import { default as default1156 } from "../icons/shield-alert";
import { default as default1157 } from "../icons/shield-ban";
import { default as default1158 } from "../icons/shield-check";
import { default as default1159 } from "../icons/shield-ellipsis";
import { default as default1160 } from "../icons/shield-half";
import { default as default1161 } from "../icons/shield-minus";
import { default as default1162 } from "../icons/shield-off";
import { default as default1163 } from "../icons/shield-user";
import { default as default1164 } from "../icons/shield-plus";
import { default as default1165 } from "../icons/shield";
import { default as default1166 } from "../icons/ship-wheel";
import { default as default1167 } from "../icons/shirt";
import { default as default1168 } from "../icons/ship";
import { default as default1169 } from "../icons/shopping-bag";
import { default as default1170 } from "../icons/shopping-cart";
import { default as default1171 } from "../icons/shopping-basket";
import { default as default1172 } from "../icons/shovel";
import { default as default1173 } from "../icons/shower-head";
import { default as default1174 } from "../icons/shredder";
import { default as default1175 } from "../icons/shrimp";
import { default as default1176 } from "../icons/shrub";
import { default as default1177 } from "../icons/shrink";
import { default as default1178 } from "../icons/shuffle";
import { default as default1179 } from "../icons/sigma";
import { default as default1180 } from "../icons/signal-high";
import { default as default1181 } from "../icons/signal-low";
import { default as default1182 } from "../icons/signal-medium";
import { default as default1183 } from "../icons/signal";
import { default as default1184 } from "../icons/signal-zero";
import { default as default1185 } from "../icons/signature";
import { default as default1186 } from "../icons/signpost-big";
import { default as default1187 } from "../icons/signpost";
import { default as default1188 } from "../icons/siren";
import { default as default1189 } from "../icons/skip-forward";
import { default as default1190 } from "../icons/skull";
import { default as default1191 } from "../icons/skip-back";
import { default as default1192 } from "../icons/slack";
import { default as default1193 } from "../icons/slash";
import { default as default1194 } from "../icons/slice";
import { default as default1195 } from "../icons/sliders-horizontal";
import { default as default1196 } from "../icons/smartphone-nfc";
import { default as default1197 } from "../icons/smartphone-charging";
import { default as default1198 } from "../icons/smartphone";
import { default as default1199 } from "../icons/smile-plus";
import { default as default1200 } from "../icons/smile";
import { default as default1201 } from "../icons/snail";
import { default as default1202 } from "../icons/snowflake";
import { default as default1203 } from "../icons/soap-dispenser-droplet";
import { default as default1204 } from "../icons/sofa";
import { default as default1205 } from "../icons/solar-panel";
import { default as default1206 } from "../icons/soup";
import { default as default1207 } from "../icons/space";
import { default as default1208 } from "../icons/sparkle";
import { default as default1209 } from "../icons/spade";
import { default as default1210 } from "../icons/speaker";
import { default as default1211 } from "../icons/speech";
import { default as default1212 } from "../icons/spell-check-2";
import { default as default1213 } from "../icons/spell-check";
import { default as default1214 } from "../icons/spline-pointer";
import { default as default1215 } from "../icons/split";
import { default as default1216 } from "../icons/spline";
import { default as default1217 } from "../icons/spool";
import { default as default1218 } from "../icons/spotlight";
import { default as default1219 } from "../icons/sprout";
import { default as default1220 } from "../icons/spray-can";
import { default as default1221 } from "../icons/square-arrow-right-enter";
import { default as default1222 } from "../icons/square-arrow-right-exit";
import { default as default1223 } from "../icons/square-dashed-bottom-code";
import { default as default1224 } from "../icons/square-dashed-bottom";
import { default as default1225 } from "../icons/square-dashed-top-solid";
import { default as default1226 } from "../icons/square-pause";
import { default as default1227 } from "../icons/square-radical";
import { default as default1228 } from "../icons/square-round-corner";
import { default as default1229 } from "../icons/square-square";
import { default as default1230 } from "../icons/square-stack";
import { default as default1231 } from "../icons/square-stop";
import { default as default1232 } from "../icons/square-star";
import { default as default1233 } from "../icons/square";
import { default as default1234 } from "../icons/squares-exclude";
import { default as default1235 } from "../icons/squares-intersect";
import { default as default1236 } from "../icons/squares-unite";
import { default as default1237 } from "../icons/squares-subtract";
import { default as default1238 } from "../icons/squircle";
import { default as default1239 } from "../icons/squircle-dashed";
import { default as default1240 } from "../icons/squirrel";
import { default as default1241 } from "../icons/stamp";
import { default as default1242 } from "../icons/star-half";
import { default as default1243 } from "../icons/star-off";
import { default as default1244 } from "../icons/step-back";
import { default as default1245 } from "../icons/star";
import { default as default1246 } from "../icons/step-forward";
import { default as default1247 } from "../icons/stethoscope";
import { default as default1248 } from "../icons/sticker";
import { default as default1249 } from "../icons/sticky-note";
import { default as default1250 } from "../icons/stone";
import { default as default1251 } from "../icons/store";
import { default as default1252 } from "../icons/stretch-horizontal";
import { default as default1253 } from "../icons/stretch-vertical";
import { default as default1254 } from "../icons/strikethrough";
import { default as default1255 } from "../icons/subscript";
import { default as default1256 } from "../icons/sun-dim";
import { default as default1257 } from "../icons/sun-medium";
import { default as default1258 } from "../icons/sun-moon";
import { default as default1259 } from "../icons/sun";
import { default as default1260 } from "../icons/sun-snow";
import { default as default1261 } from "../icons/sunrise";
import { default as default1262 } from "../icons/sunset";
import { default as default1263 } from "../icons/superscript";
import { default as default1264 } from "../icons/swatch-book";
import { default as default1265 } from "../icons/swiss-franc";
import { default as default1266 } from "../icons/sword";
import { default as default1267 } from "../icons/switch-camera";
import { default as default1268 } from "../icons/syringe";
import { default as default1269 } from "../icons/swords";
import { default as default1270 } from "../icons/table-cells-merge";
import { default as default1271 } from "../icons/table-2";
import { default as default1272 } from "../icons/table-cells-split";
import { default as default1273 } from "../icons/table-columns-split";
import { default as default1274 } from "../icons/table-of-contents";
import { default as default1275 } from "../icons/table-properties";
import { default as default1276 } from "../icons/table";
import { default as default1277 } from "../icons/tablet-smartphone";
import { default as default1278 } from "../icons/table-rows-split";
import { default as default1279 } from "../icons/tablet";
import { default as default1280 } from "../icons/tablets";
import { default as default1281 } from "../icons/tag";
import { default as default1282 } from "../icons/tally-1";
import { default as default1283 } from "../icons/tags";
import { default as default1284 } from "../icons/tally-3";
import { default as default1285 } from "../icons/tally-2";
import { default as default1286 } from "../icons/tally-4";
import { default as default1287 } from "../icons/tally-5";
import { default as default1288 } from "../icons/tangent";
import { default as default1289 } from "../icons/target";
import { default as default1290 } from "../icons/telescope";
import { default as default1291 } from "../icons/tent-tree";
import { default as default1292 } from "../icons/tent";
import { default as default1293 } from "../icons/terminal";
import { default as default1294 } from "../icons/test-tube";
import { default as default1295 } from "../icons/test-tubes";
import { default as default1296 } from "../icons/text-cursor-input";
import { default as default1297 } from "../icons/text-cursor";
import { default as default1298 } from "../icons/text-quote";
import { default as default1299 } from "../icons/text-search";
import { default as default1300 } from "../icons/theater";
import { default as default1301 } from "../icons/thermometer-snowflake";
import { default as default1302 } from "../icons/thermometer";
import { default as default1303 } from "../icons/thermometer-sun";
import { default as default1304 } from "../icons/thumbs-down";
import { default as default1305 } from "../icons/ticket-check";
import { default as default1306 } from "../icons/thumbs-up";
import { default as default1307 } from "../icons/ticket-minus";
import { default as default1308 } from "../icons/ticket-percent";
import { default as default1309 } from "../icons/ticket-plus";
import { default as default1310 } from "../icons/ticket-slash";
import { default as default1311 } from "../icons/ticket-x";
import { default as default1312 } from "../icons/tickets-plane";
import { default as default1313 } from "../icons/ticket";
import { default as default1314 } from "../icons/tickets";
import { default as default1315 } from "../icons/timer-off";
import { default as default1316 } from "../icons/timer-reset";
import { default as default1317 } from "../icons/timer";
import { default as default1318 } from "../icons/toggle-left";
import { default as default1319 } from "../icons/toggle-right";
import { default as default1320 } from "../icons/toilet";
import { default as default1321 } from "../icons/tool-case";
import { default as default1322 } from "../icons/toolbox";
import { default as default1323 } from "../icons/tornado";
import { default as default1324 } from "../icons/torus";
import { default as default1325 } from "../icons/touchpad-off";
import { default as default1326 } from "../icons/towel-rack";
import { default as default1327 } from "../icons/touchpad";
import { default as default1328 } from "../icons/tower-control";
import { default as default1329 } from "../icons/toy-brick";
import { default as default1330 } from "../icons/tractor";
import { default as default1331 } from "../icons/traffic-cone";
import { default as default1332 } from "../icons/train-front-tunnel";
import { default as default1333 } from "../icons/train-track";
import { default as default1334 } from "../icons/trash-2";
import { default as default1335 } from "../icons/train-front";
import { default as default1336 } from "../icons/transgender";
import { default as default1337 } from "../icons/trash";
import { default as default1338 } from "../icons/tree-deciduous";
import { default as default1339 } from "../icons/tree-pine";
import { default as default1340 } from "../icons/trees";
import { default as default1341 } from "../icons/trello";
import { default as default1342 } from "../icons/trending-down";
import { default as default1343 } from "../icons/trending-up-down";
import { default as default1344 } from "../icons/triangle-right";
import { default as default1345 } from "../icons/triangle-dashed";
import { default as default1346 } from "../icons/trending-up";
import { default as default1347 } from "../icons/truck-electric";
import { default as default1348 } from "../icons/trophy";
import { default as default1349 } from "../icons/triangle";
import { default as default1350 } from "../icons/truck";
import { default as default1351 } from "../icons/turkish-lira";
import { default as default1352 } from "../icons/turntable";
import { default as default1353 } from "../icons/turtle";
import { default as default1354 } from "../icons/tv-minimal-play";
import { default as default1355 } from "../icons/twitch";
import { default as default1356 } from "../icons/tv";
import { default as default1357 } from "../icons/twitter";
import { default as default1358 } from "../icons/type-outline";
import { default as default1359 } from "../icons/type";
import { default as default1360 } from "../icons/umbrella-off";
import { default as default1361 } from "../icons/umbrella";
import { default as default1362 } from "../icons/underline";
import { default as default1363 } from "../icons/undo-2";
import { default as default1364 } from "../icons/undo-dot";
import { default as default1365 } from "../icons/undo";
import { default as default1366 } from "../icons/unfold-vertical";
import { default as default1367 } from "../icons/unfold-horizontal";
import { default as default1368 } from "../icons/ungroup";
import { default as default1369 } from "../icons/unlink-2";
import { default as default1370 } from "../icons/unlink";
import { default as default1371 } from "../icons/unplug";
import { default as default1372 } from "../icons/usb";
import { default as default1373 } from "../icons/upload";
import { default as default1374 } from "../icons/user-check";
import { default as default1375 } from "../icons/user-key";
import { default as default1376 } from "../icons/user-cog";
import { default as default1377 } from "../icons/user-lock";
import { default as default1378 } from "../icons/user-pen";
import { default as default1379 } from "../icons/user-minus";
import { default as default1380 } from "../icons/user-plus";
import { default as default1381 } from "../icons/user-round-key";
import { default as default1382 } from "../icons/user-round-pen";
import { default as default1383 } from "../icons/user-round-search";
import { default as default1384 } from "../icons/user-search";
import { default as default1385 } from "../icons/user-star";
import { default as default1386 } from "../icons/user-x";
import { default as default1387 } from "../icons/user";
import { default as default1388 } from "../icons/users";
import { default as default1389 } from "../icons/utility-pole";
import { default as default1390 } from "../icons/van";
import { default as default1391 } from "../icons/vault";
import { default as default1392 } from "../icons/variable";
import { default as default1393 } from "../icons/vector-square";
import { default as default1394 } from "../icons/venetian-mask";
import { default as default1395 } from "../icons/vegan";
import { default as default1396 } from "../icons/venus";
import { default as default1397 } from "../icons/venus-and-mars";
import { default as default1398 } from "../icons/vibrate-off";
import { default as default1399 } from "../icons/vibrate";
import { default as default1400 } from "../icons/video-off";
import { default as default1401 } from "../icons/videotape";
import { default as default1402 } from "../icons/video";
import { default as default1403 } from "../icons/view";
import { default as default1404 } from "../icons/voicemail";
import { default as default1405 } from "../icons/volleyball";
import { default as default1406 } from "../icons/volume-1";
import { default as default1407 } from "../icons/volume-2";
import { default as default1408 } from "../icons/volume-off";
import { default as default1409 } from "../icons/volume-x";
import { default as default1410 } from "../icons/volume";
import { default as default1411 } from "../icons/vote";
import { default as default1412 } from "../icons/wallet-cards";
import { default as default1413 } from "../icons/wallet";
import { default as default1414 } from "../icons/wallpaper";
import { default as default1415 } from "../icons/wand";
import { default as default1416 } from "../icons/warehouse";
import { default as default1417 } from "../icons/washing-machine";
import { default as default1418 } from "../icons/watch";
import { default as default1419 } from "../icons/waves-arrow-down";
import { default as default1420 } from "../icons/waves-arrow-up";
import { default as default1421 } from "../icons/waves-ladder";
import { default as default1422 } from "../icons/waves";
import { default as default1423 } from "../icons/waypoints";
import { default as default1424 } from "../icons/webcam";
import { default as default1425 } from "../icons/webhook-off";
import { default as default1426 } from "../icons/weight-tilde";
import { default as default1427 } from "../icons/webhook";
import { default as default1428 } from "../icons/weight";
import { default as default1429 } from "../icons/wheat";
import { default as default1430 } from "../icons/wheat-off";
import { default as default1431 } from "../icons/whole-word";
import { default as default1432 } from "../icons/wifi-cog";
import { default as default1433 } from "../icons/wifi-high";
import { default as default1434 } from "../icons/wifi-low";
import { default as default1435 } from "../icons/wifi-off";
import { default as default1436 } from "../icons/wifi-zero";
import { default as default1437 } from "../icons/wifi-pen";
import { default as default1438 } from "../icons/wifi-sync";
import { default as default1439 } from "../icons/wifi";
import { default as default1440 } from "../icons/wind-arrow-down";
import { default as default1441 } from "../icons/wine-off";
import { default as default1442 } from "../icons/wine";
import { default as default1443 } from "../icons/wind";
import { default as default1444 } from "../icons/worm";
import { default as default1445 } from "../icons/workflow";
import { default as default1446 } from "../icons/wrench";
import { default as default1447 } from "../icons/x-line-top";
import { default as default1448 } from "../icons/x";
import { default as default1449 } from "../icons/youtube";
import { default as default1450 } from "../icons/zap-off";
import { default as default1451 } from "../icons/zap";
import { default as default1452 } from "../icons/zodiac-aquarius";
import { default as default1453 } from "../icons/zodiac-aries";
import { default as default1454 } from "../icons/zodiac-capricorn";
import { default as default1455 } from "../icons/zodiac-cancer";
import { default as default1456 } from "../icons/zodiac-leo";
import { default as default1457 } from "../icons/zodiac-gemini";
import { default as default1458 } from "../icons/zodiac-libra";
import { default as default1459 } from "../icons/zodiac-pisces";
import { default as default1460 } from "../icons/zodiac-ophiuchus";
import { default as default1461 } from "../icons/zodiac-sagittarius";
import { default as default1462 } from "../icons/zodiac-scorpio";
import { default as default1463 } from "../icons/zodiac-virgo";
import { default as default1464 } from "../icons/zodiac-taurus";
import { default as default1465 } from "../icons/zoom-in";
import { default as default1466 } from "../icons/zoom-out";
import { default as default1467 } from "../icons/fingerprint-pattern";
import {
  default as default1468
} from "../icons/fingerprint-pattern";
import { default as default1469 } from "../icons/arrow-down-0-1";
import { default as default1470 } from "../icons/arrow-down-1-0";
import { default as default1471 } from "../icons/arrow-up-0-1";
import { default as default1472 } from "../icons/arrow-up-1-0";
import { default as default1473 } from "../icons/alarm-clock-check";
import {
  default as default1474
} from "../icons/alarm-check";
import { default as default1475 } from "../icons/alarm-clock-minus";
import {
  default as default1476
} from "../icons/alarm-minus";
import { default as default1477 } from "../icons/alarm-clock-plus";
import {
  default as default1478
} from "../icons/alarm-plus";
import { default as default1479 } from "../icons/arrow-down-a-z";
import {
  default as default1480
} from "../icons/arrow-down-az";
import { default as default1481 } from "../icons/arrow-down-wide-narrow";
import {
  default as default1482
} from "../icons/sort-desc";
import { default as default1483 } from "../icons/arrow-down-z-a";
import {
  default as default1484
} from "../icons/arrow-down-za";
import { default as default1485 } from "../icons/arrow-up-a-z";
import {
  default as default1486
} from "../icons/arrow-up-az";
import { default as default1487 } from "../icons/arrow-up-narrow-wide";
import {
  default as default1488
} from "../icons/sort-asc";
import { default as default1489 } from "../icons/arrow-up-z-a";
import {
  default as default1490
} from "../icons/arrow-up-za";
import { default as default1491 } from "../icons/axis-3d";
import {
  default as default1492
} from "../icons/axis-3-d";
import { default as default1493 } from "../icons/badge-check";
import {
  default as default1494
} from "../icons/verified";
import { default as default1495 } from "../icons/badge-question-mark";
import {
  default as default1496
} from "../icons/badge-help";
import { default as default1497 } from "../icons/between-horizontal-end";
import {
  default as default1498
} from "../icons/between-horizonal-end";
import { default as default1499 } from "../icons/between-horizontal-start";
import {
  default as default1500
} from "../icons/between-horizonal-start";
import { default as default1501 } from "../icons/book-dashed";
import {
  default as default1502
} from "../icons/book-template";
import { default as default1503 } from "../icons/braces";
import {
  default as default1504
} from "../icons/curly-braces";
import { default as default1505 } from "../icons/captions";
import {
  default as default1506
} from "../icons/subtitles";
import { default as default1507 } from "../icons/chart-area";
import {
  default as default1508
} from "../icons/area-chart";
import { default as default1509 } from "../icons/chart-bar-big";
import {
  default as default1510
} from "../icons/bar-chart-horizontal-big";
import { default as default1511 } from "../icons/chart-bar";
import {
  default as default1512
} from "../icons/bar-chart-horizontal";
import { default as default1513 } from "../icons/chart-column-big";
import {
  default as default1514
} from "../icons/bar-chart-big";
import { default as default1515 } from "../icons/chart-candlestick";
import {
  default as default1516
} from "../icons/candlestick-chart";
import { default as default1517 } from "../icons/chart-column-increasing";
import {
  default as default1518
} from "../icons/bar-chart-4";
import { default as default1519 } from "../icons/chart-column";
import {
  default as default1520
} from "../icons/bar-chart-3";
import { default as default1521 } from "../icons/chart-line";
import {
  default as default1522
} from "../icons/line-chart";
import { default as default1523 } from "../icons/chart-no-axes-column-increasing";
import {
  default as default1524
} from "../icons/bar-chart";
import { default as default1525 } from "../icons/chart-no-axes-column";
import {
  default as default1526
} from "../icons/bar-chart-2";
import { default as default1527 } from "../icons/chart-no-axes-gantt";
import {
  default as default1528
} from "../icons/gantt-chart";
import { default as default1529 } from "../icons/chart-pie";
import {
  default as default1530
} from "../icons/pie-chart";
import { default as default1531 } from "../icons/chart-scatter";
import {
  default as default1532
} from "../icons/scatter-chart";
import { default as default1533 } from "../icons/chromium";
import {
  default as default1534
} from "../icons/chrome";
import { default as default1535 } from "../icons/circle-arrow-down";
import {
  default as default1536
} from "../icons/arrow-down-circle";
import { default as default1537 } from "../icons/circle-arrow-out-down-right";
import {
  default as default1538
} from "../icons/arrow-down-right-from-circle";
import { default as default1539 } from "../icons/circle-alert";
import {
  default as default1540
} from "../icons/alert-circle";
import { default as default1541 } from "../icons/circle-arrow-left";
import {
  default as default1542
} from "../icons/arrow-left-circle";
import { default as default1543 } from "../icons/circle-arrow-out-down-left";
import {
  default as default1544
} from "../icons/arrow-down-left-from-circle";
import { default as default1545 } from "../icons/circle-arrow-out-up-left";
import {
  default as default1546
} from "../icons/arrow-up-left-from-circle";
import { default as default1547 } from "../icons/circle-arrow-out-up-right";
import {
  default as default1548
} from "../icons/arrow-up-right-from-circle";
import { default as default1549 } from "../icons/circle-arrow-up";
import {
  default as default1550
} from "../icons/arrow-up-circle";
import { default as default1551 } from "../icons/circle-check-big";
import {
  default as default1552
} from "../icons/check-circle";
import { default as default1553 } from "../icons/circle-check";
import {
  default as default1554
} from "../icons/check-circle-2";
import { default as default1555 } from "../icons/circle-arrow-right";
import {
  default as default1556
} from "../icons/arrow-right-circle";
import { default as default1557 } from "../icons/circle-chevron-down";
import {
  default as default1558
} from "../icons/chevron-down-circle";
import { default as default1559 } from "../icons/circle-chevron-left";
import {
  default as default1560
} from "../icons/chevron-left-circle";
import { default as default1561 } from "../icons/circle-chevron-right";
import {
  default as default1562
} from "../icons/chevron-right-circle";
import { default as default1563 } from "../icons/circle-chevron-up";
import {
  default as default1564
} from "../icons/chevron-up-circle";
import { default as default1565 } from "../icons/circle-divide";
import {
  default as default1566
} from "../icons/divide-circle";
import { default as default1567 } from "../icons/circle-gauge";
import {
  default as default1568
} from "../icons/gauge-circle";
import { default as default1569 } from "../icons/circle-minus";
import {
  default as default1570
} from "../icons/minus-circle";
import { default as default1571 } from "../icons/circle-parking-off";
import {
  default as default1572
} from "../icons/parking-circle-off";
import { default as default1573 } from "../icons/circle-pause";
import {
  default as default1574
} from "../icons/pause-circle";
import { default as default1575 } from "../icons/circle-parking";
import {
  default as default1576
} from "../icons/parking-circle";
import { default as default1577 } from "../icons/circle-percent";
import {
  default as default1578
} from "../icons/percent-circle";
import { default as default1579 } from "../icons/circle-play";
import {
  default as default1580
} from "../icons/play-circle";
import { default as default1581 } from "../icons/circle-plus";
import {
  default as default1582
} from "../icons/plus-circle";
import { default as default1583 } from "../icons/circle-question-mark";
import {
  default as default1584
} from "../icons/help-circle";
import {
  default as default1585
} from "../icons/circle-help";
import { default as default1586 } from "../icons/circle-power";
import {
  default as default1587
} from "../icons/power-circle";
import { default as default1588 } from "../icons/circle-slash-2";
import {
  default as default1589
} from "../icons/circle-slashed";
import { default as default1590 } from "../icons/circle-stop";
import {
  default as default1591
} from "../icons/stop-circle";
import { default as default1592 } from "../icons/circle-user";
import {
  default as default1593
} from "../icons/user-circle";
import { default as default1594 } from "../icons/circle-user-round";
import {
  default as default1595
} from "../icons/user-circle-2";
import { default as default1596 } from "../icons/circle-x";
import {
  default as default1597
} from "../icons/x-circle";
import { default as default1598 } from "../icons/clipboard-pen";
import {
  default as default1599
} from "../icons/clipboard-edit";
import { default as default1600 } from "../icons/clipboard-pen-line";
import {
  default as default1601
} from "../icons/clipboard-signature";
import { default as default1602 } from "../icons/cloud-download";
import {
  default as default1603
} from "../icons/download-cloud";
import { default as default1604 } from "../icons/cloud-upload";
import {
  default as default1605
} from "../icons/upload-cloud";
import { default as default1606 } from "../icons/code-xml";
import {
  default as default1607
} from "../icons/code-2";
import { default as default1608 } from "../icons/columns-2";
import {
  default as default1609
} from "../icons/columns";
import { default as default1610 } from "../icons/columns-3-cog";
import {
  default as default1611
} from "../icons/columns-settings";
import {
  default as default1612
} from "../icons/table-config";
import { default as default1613 } from "../icons/columns-3";
import {
  default as default1614
} from "../icons/panels-left-right";
import { default as default1615 } from "../icons/contact-round";
import {
  default as default1616
} from "../icons/contact-2";
import { default as default1617 } from "../icons/diamond-percent";
import {
  default as default1618
} from "../icons/percent-diamond";
import { default as default1619 } from "../icons/earth";
import {
  default as default1620
} from "../icons/globe-2";
import { default as default1621 } from "../icons/ellipsis-vertical";
import {
  default as default1622
} from "../icons/more-vertical";
import { default as default1623 } from "../icons/ellipsis";
import {
  default as default1624
} from "../icons/more-horizontal";
import { default as default1625 } from "../icons/file-axis-3d";
import {
  default as default1626
} from "../icons/file-axis-3-d";
import { default as default1627 } from "../icons/file-braces";
import {
  default as default1628
} from "../icons/file-json";
import { default as default1629 } from "../icons/file-braces-corner";
import {
  default as default1630
} from "../icons/file-json-2";
import { default as default1631 } from "../icons/file-badge";
import {
  default as default1632
} from "../icons/file-badge-2";
import { default as default1633 } from "../icons/file-chart-column";
import {
  default as default1634
} from "../icons/file-bar-chart-2";
import { default as default1635 } from "../icons/file-chart-column-increasing";
import {
  default as default1636
} from "../icons/file-bar-chart";
import { default as default1637 } from "../icons/file-chart-line";
import {
  default as default1638
} from "../icons/file-line-chart";
import { default as default1639 } from "../icons/file-chart-pie";
import {
  default as default1640
} from "../icons/file-pie-chart";
import { default as default1641 } from "../icons/file-check-corner";
import {
  default as default1642
} from "../icons/file-check-2";
import { default as default1643 } from "../icons/file-code-corner";
import {
  default as default1644
} from "../icons/file-code-2";
import { default as default1645 } from "../icons/file-exclamation-point";
import {
  default as default1646
} from "../icons/file-warning";
import { default as default1647 } from "../icons/file-cog";
import {
  default as default1648
} from "../icons/file-cog-2";
import { default as default1649 } from "../icons/file-headphone";
import {
  default as default1650
} from "../icons/file-audio";
import {
  default as default1651
} from "../icons/file-audio-2";
import { default as default1652 } from "../icons/file-lock";
import {
  default as default1653
} from "../icons/file-lock-2";
import { default as default1654 } from "../icons/file-key";
import {
  default as default1655
} from "../icons/file-key-2";
import { default as default1656 } from "../icons/file-minus-corner";
import {
  default as default1657
} from "../icons/file-minus-2";
import { default as default1658 } from "../icons/file-pen-line";
import {
  default as default1659
} from "../icons/file-signature";
import { default as default1660 } from "../icons/file-pen";
import {
  default as default1661
} from "../icons/file-edit";
import { default as default1662 } from "../icons/file-play";
import {
  default as default1663
} from "../icons/file-video";
import { default as default1664 } from "../icons/file-plus-corner";
import {
  default as default1665
} from "../icons/file-plus-2";
import { default as default1666 } from "../icons/file-question-mark";
import {
  default as default1667
} from "../icons/file-question";
import { default as default1668 } from "../icons/file-search-corner";
import {
  default as default1669
} from "../icons/file-search-2";
import { default as default1670 } from "../icons/file-signal";
import {
  default as default1671
} from "../icons/file-volume-2";
import { default as default1672 } from "../icons/file-type-corner";
import {
  default as default1673
} from "../icons/file-type-2";
import { default as default1674 } from "../icons/file-video-camera";
import {
  default as default1675
} from "../icons/file-video-2";
import { default as default1676 } from "../icons/file-x-corner";
import {
  default as default1677
} from "../icons/file-x-2";
import { default as default1678 } from "../icons/folder-cog";
import {
  default as default1679
} from "../icons/folder-cog-2";
import { default as default1680 } from "../icons/folder-pen";
import {
  default as default1681
} from "../icons/folder-edit";
import { default as default1682 } from "../icons/funnel-x";
import {
  default as default1683
} from "../icons/filter-x";
import { default as default1684 } from "../icons/funnel";
import {
  default as default1685
} from "../icons/filter";
import { default as default1686 } from "../icons/git-commit-horizontal";
import {
  default as default1687
} from "../icons/git-commit";
import { default as default1688 } from "../icons/grid-2x2-plus";
import {
  default as default1689
} from "../icons/grid-2-x-2-plus";
import { default as default1690 } from "../icons/grid-2x2-check";
import {
  default as default1691
} from "../icons/grid-2-x-2-check";
import { default as default1692 } from "../icons/grid-2x2";
import {
  default as default1693
} from "../icons/grid-2-x-2";
import { default as default1694 } from "../icons/grid-2x2-x";
import {
  default as default1695
} from "../icons/grid-2-x-2-x";
import { default as default1696 } from "../icons/grid-3x3";
import {
  default as default1697
} from "../icons/grid";
import {
  default as default1698
} from "../icons/grid-3-x-3";
import { default as default1699 } from "../icons/hand-grab";
import {
  default as default1700
} from "../icons/grab";
import { default as default1701 } from "../icons/hand-helping";
import {
  default as default1702
} from "../icons/helping-hand";
import { default as default1703 } from "../icons/house";
import {
  default as default1704
} from "../icons/home";
import { default as default1705 } from "../icons/ice-cream-bowl";
import {
  default as default1706
} from "../icons/ice-cream-2";
import { default as default1707 } from "../icons/ice-cream-cone";
import {
  default as default1708
} from "../icons/ice-cream";
import { default as default1709 } from "../icons/laptop-minimal";
import {
  default as default1710
} from "../icons/laptop-2";
import { default as default1711 } from "../icons/layers";
import {
  default as default1712
} from "../icons/layers-3";
import { default as default1713 } from "../icons/list-indent-decrease";
import {
  default as default1714
} from "../icons/outdent";
import {
  default as default1715
} from "../icons/indent-decrease";
import { default as default1716 } from "../icons/list-indent-increase";
import {
  default as default1717
} from "../icons/indent";
import {
  default as default1718
} from "../icons/indent-increase";
import { default as default1719 } from "../icons/loader-circle";
import {
  default as default1720
} from "../icons/loader-2";
import { default as default1721 } from "../icons/lock-keyhole-open";
import {
  default as default1722
} from "../icons/unlock-keyhole";
import { default as default1723 } from "../icons/mail-question-mark";
import {
  default as default1724
} from "../icons/mail-question";
import { default as default1725 } from "../icons/lock-open";
import {
  default as default1726
} from "../icons/unlock";
import { default as default1727 } from "../icons/map-pin-pen";
import {
  default as default1728
} from "../icons/location-edit";
import { default as default1729 } from "../icons/message-circle-question-mark";
import {
  default as default1730
} from "../icons/message-circle-question";
import { default as default1731 } from "../icons/mic-vocal";
import {
  default as default1732
} from "../icons/mic-2";
import { default as default1733 } from "../icons/move-3d";
import {
  default as default1734
} from "../icons/move-3-d";
import { default as default1735 } from "../icons/octagon-alert";
import {
  default as default1736
} from "../icons/alert-octagon";
import { default as default1737 } from "../icons/octagon-pause";
import {
  default as default1738
} from "../icons/pause-octagon";
import { default as default1739 } from "../icons/octagon-x";
import {
  default as default1740
} from "../icons/x-octagon";
import { default as default1741 } from "../icons/paintbrush-vertical";
import {
  default as default1742
} from "../icons/paintbrush-2";
import { default as default1743 } from "../icons/panel-bottom-dashed";
import {
  default as default1744
} from "../icons/panel-bottom-inactive";
import { default as default1745 } from "../icons/panel-left-close";
import {
  default as default1746
} from "../icons/sidebar-close";
import { default as default1747 } from "../icons/panel-left-dashed";
import {
  default as default1748
} from "../icons/panel-left-inactive";
import { default as default1749 } from "../icons/panel-left-open";
import {
  default as default1750
} from "../icons/sidebar-open";
import { default as default1751 } from "../icons/panel-right-dashed";
import {
  default as default1752
} from "../icons/panel-right-inactive";
import { default as default1753 } from "../icons/panel-left";
import {
  default as default1754
} from "../icons/sidebar";
import { default as default1755 } from "../icons/panel-top-dashed";
import {
  default as default1756
} from "../icons/panel-top-inactive";
import { default as default1757 } from "../icons/panels-top-left";
import {
  default as default1758
} from "../icons/layout";
import { default as default1759 } from "../icons/pen-line";
import {
  default as default1760
} from "../icons/edit-3";
import { default as default1761 } from "../icons/pen";
import {
  default as default1762
} from "../icons/edit-2";
import { default as default1763 } from "../icons/plug-zap";
import {
  default as default1764
} from "../icons/plug-zap-2";
import { default as default1765 } from "../icons/rectangle-ellipsis";
import {
  default as default1766
} from "../icons/form-input";
import { default as default1767 } from "../icons/rotate-3d";
import {
  default as default1768
} from "../icons/rotate-3-d";
import { default as default1769 } from "../icons/rows-3";
import {
  default as default1770
} from "../icons/panels-top-bottom";
import { default as default1771 } from "../icons/rows-2";
import {
  default as default1772
} from "../icons/rows";
import { default as default1773 } from "../icons/scale-3d";
import {
  default as default1774
} from "../icons/scale-3-d";
import { default as default1775 } from "../icons/send-horizontal";
import {
  default as default1776
} from "../icons/send-horizonal";
import { default as default1777 } from "../icons/shield-question-mark";
import {
  default as default1778
} from "../icons/shield-question";
import { default as default1779 } from "../icons/shield-x";
import {
  default as default1780
} from "../icons/shield-close";
import { default as default1781 } from "../icons/sliders-vertical";
import {
  default as default1782
} from "../icons/sliders";
import { default as default1783 } from "../icons/sparkles";
import {
  default as default1784
} from "../icons/stars";
import { default as default1785 } from "../icons/square-activity";
import {
  default as default1786
} from "../icons/activity-square";
import { default as default1787 } from "../icons/square-arrow-down-left";
import {
  default as default1788
} from "../icons/arrow-down-left-square";
import { default as default1789 } from "../icons/square-arrow-down-right";
import {
  default as default1790
} from "../icons/arrow-down-right-square";
import { default as default1791 } from "../icons/square-arrow-left";
import {
  default as default1792
} from "../icons/arrow-left-square";
import { default as default1793 } from "../icons/square-arrow-out-down-left";
import {
  default as default1794
} from "../icons/arrow-down-left-from-square";
import { default as default1795 } from "../icons/square-arrow-down";
import {
  default as default1796
} from "../icons/arrow-down-square";
import { default as default1797 } from "../icons/square-arrow-out-up-left";
import {
  default as default1798
} from "../icons/arrow-up-left-from-square";
import { default as default1799 } from "../icons/square-arrow-out-down-right";
import {
  default as default1800
} from "../icons/arrow-down-right-from-square";
import { default as default1801 } from "../icons/square-arrow-out-up-right";
import {
  default as default1802
} from "../icons/arrow-up-right-from-square";
import { default as default1803 } from "../icons/square-arrow-right";
import {
  default as default1804
} from "../icons/arrow-right-square";
import { default as default1805 } from "../icons/square-arrow-up-left";
import {
  default as default1806
} from "../icons/arrow-up-left-square";
import { default as default1807 } from "../icons/square-arrow-up-right";
import {
  default as default1808
} from "../icons/arrow-up-right-square";
import { default as default1809 } from "../icons/square-asterisk";
import {
  default as default1810
} from "../icons/asterisk-square";
import { default as default1811 } from "../icons/square-arrow-up";
import {
  default as default1812
} from "../icons/arrow-up-square";
import { default as default1813 } from "../icons/square-centerline-dashed-horizontal";
import {
  default as default1814
} from "../icons/flip-horizontal";
import { default as default1815 } from "../icons/square-bottom-dashed-scissors";
import {
  default as default1816
} from "../icons/scissors-square-dashed-bottom";
import { default as default1817 } from "../icons/square-centerline-dashed-vertical";
import {
  default as default1818
} from "../icons/flip-vertical";
import { default as default1819 } from "../icons/square-check-big";
import {
  default as default1820
} from "../icons/check-square";
import { default as default1821 } from "../icons/square-chart-gantt";
import {
  default as default1822
} from "../icons/gantt-chart-square";
import {
  default as default1823
} from "../icons/square-gantt-chart";
import { default as default1824 } from "../icons/square-check";
import {
  default as default1825
} from "../icons/check-square-2";
import { default as default1826 } from "../icons/square-chevron-left";
import {
  default as default1827
} from "../icons/chevron-left-square";
import { default as default1828 } from "../icons/square-chevron-down";
import {
  default as default1829
} from "../icons/chevron-down-square";
import { default as default1830 } from "../icons/square-chevron-right";
import {
  default as default1831
} from "../icons/chevron-right-square";
import { default as default1832 } from "../icons/square-chevron-up";
import {
  default as default1833
} from "../icons/chevron-up-square";
import { default as default1834 } from "../icons/square-code";
import {
  default as default1835
} from "../icons/code-square";
import { default as default1836 } from "../icons/square-dashed-kanban";
import {
  default as default1837
} from "../icons/kanban-square-dashed";
import { default as default1838 } from "../icons/square-dashed-mouse-pointer";
import {
  default as default1839
} from "../icons/mouse-pointer-square-dashed";
import { default as default1840 } from "../icons/square-dashed";
import {
  default as default1841
} from "../icons/box-select";
import { default as default1842 } from "../icons/square-divide";
import {
  default as default1843
} from "../icons/divide-square";
import { default as default1844 } from "../icons/square-dot";
import {
  default as default1845
} from "../icons/dot-square";
import { default as default1846 } from "../icons/square-equal";
import {
  default as default1847
} from "../icons/equal-square";
import { default as default1848 } from "../icons/square-function";
import {
  default as default1849
} from "../icons/function-square";
import { default as default1850 } from "../icons/square-kanban";
import {
  default as default1851
} from "../icons/kanban-square";
import { default as default1852 } from "../icons/square-library";
import {
  default as default1853
} from "../icons/library-square";
import { default as default1854 } from "../icons/square-m";
import {
  default as default1855
} from "../icons/m-square";
import { default as default1856 } from "../icons/square-menu";
import {
  default as default1857
} from "../icons/menu-square";
import { default as default1858 } from "../icons/square-mouse-pointer";
import {
  default as default1859
} from "../icons/inspect";
import { default as default1860 } from "../icons/square-parking-off";
import {
  default as default1861
} from "../icons/parking-square-off";
import { default as default1862 } from "../icons/square-minus";
import {
  default as default1863
} from "../icons/minus-square";
import { default as default1864 } from "../icons/square-parking";
import {
  default as default1865
} from "../icons/parking-square";
import { default as default1866 } from "../icons/square-pen";
import {
  default as default1867
} from "../icons/pen-box";
import {
  default as default1868
} from "../icons/edit";
import {
  default as default1869
} from "../icons/pen-square";
import { default as default1870 } from "../icons/square-pi";
import {
  default as default1871
} from "../icons/pi-square";
import { default as default1872 } from "../icons/square-percent";
import {
  default as default1873
} from "../icons/percent-square";
import { default as default1874 } from "../icons/square-pilcrow";
import {
  default as default1875
} from "../icons/pilcrow-square";
import { default as default1876 } from "../icons/square-play";
import {
  default as default1877
} from "../icons/play-square";
import { default as default1878 } from "../icons/square-power";
import {
  default as default1879
} from "../icons/power-square";
import { default as default1880 } from "../icons/square-scissors";
import {
  default as default1881
} from "../icons/scissors-square";
import { default as default1882 } from "../icons/square-plus";
import {
  default as default1883
} from "../icons/plus-square";
import { default as default1884 } from "../icons/square-sigma";
import {
  default as default1885
} from "../icons/sigma-square";
import { default as default1886 } from "../icons/square-split-horizontal";
import {
  default as default1887
} from "../icons/split-square-horizontal";
import { default as default1888 } from "../icons/square-slash";
import {
  default as default1889
} from "../icons/slash-square";
import { default as default1890 } from "../icons/square-split-vertical";
import {
  default as default1891
} from "../icons/split-square-vertical";
import { default as default1892 } from "../icons/square-user-round";
import {
  default as default1893
} from "../icons/user-square-2";
import { default as default1894 } from "../icons/square-user";
import {
  default as default1895
} from "../icons/user-square";
import { default as default1896 } from "../icons/square-terminal";
import {
  default as default1897
} from "../icons/terminal-square";
import { default as default1898 } from "../icons/square-x";
import {
  default as default1899
} from "../icons/x-square";
import { default as default1900 } from "../icons/test-tube-diagonal";
import {
  default as default1901
} from "../icons/test-tube-2";
import { default as default1902 } from "../icons/text-align-center";
import {
  default as default1903
} from "../icons/align-center";
import { default as default1904 } from "../icons/text-align-end";
import {
  default as default1905
} from "../icons/align-right";
import { default as default1906 } from "../icons/text-align-justify";
import {
  default as default1907
} from "../icons/align-justify";
import { default as default1908 } from "../icons/text-align-start";
import {
  default as default1909
} from "../icons/text";
import {
  default as default1910
} from "../icons/align-left";
import { default as default1911 } from "../icons/text-initial";
import {
  default as default1912
} from "../icons/letter-text";
import { default as default1913 } from "../icons/text-select";
import {
  default as default1914
} from "../icons/text-selection";
import { default as default1915 } from "../icons/text-wrap";
import {
  default as default1916
} from "../icons/wrap-text";
import { default as default1917 } from "../icons/tram-front";
import {
  default as default1918
} from "../icons/train";
import { default as default1919 } from "../icons/tree-palm";
import {
  default as default1920
} from "../icons/palmtree";
import { default as default1921 } from "../icons/triangle-alert";
import {
  default as default1922
} from "../icons/alert-triangle";
import { default as default1923 } from "../icons/tv-minimal";
import {
  default as default1924
} from "../icons/tv-2";
import { default as default1925 } from "../icons/university";
import {
  default as default1926
} from "../icons/school-2";
import { default as default1927 } from "../icons/user-round-check";
import {
  default as default1928
} from "../icons/user-check-2";
import { default as default1929 } from "../icons/user-round-cog";
import {
  default as default1930
} from "../icons/user-cog-2";
import { default as default1931 } from "../icons/user-round-minus";
import {
  default as default1932
} from "../icons/user-minus-2";
import { default as default1933 } from "../icons/user-round-plus";
import {
  default as default1934
} from "../icons/user-plus-2";
import { default as default1935 } from "../icons/user-round-x";
import {
  default as default1936
} from "../icons/user-x-2";
import { default as default1937 } from "../icons/user-round";
import {
  default as default1938
} from "../icons/user-2";
import { default as default1939 } from "../icons/users-round";
import {
  default as default1940
} from "../icons/users-2";
import { default as default1941 } from "../icons/utensils";
import {
  default as default1942
} from "../icons/fork-knife";
import { default as default1943 } from "../icons/utensils-crossed";
import {
  default as default1944
} from "../icons/fork-knife-crossed";
import { default as default1945 } from "../icons/wand-sparkles";
import {
  default as default1946
} from "../icons/wand-2";
import { default as default1947 } from "../icons/wallet-minimal";
import {
  default as default1948
} from "../icons/wallet-2";
export {
  default2 as AArrowDownIcon,
  default3 as AArrowUpIcon,
  default4 as ALargeSmallIcon,
  default5 as AccessibilityIcon,
  default6 as ActivityIcon,
  default1786 as ActivitySquareIcon,
  default7 as AirVentIcon,
  default8 as AirplayIcon,
  default1474 as AlarmCheckIcon,
  default1473 as AlarmClockCheckIcon,
  default10 as AlarmClockIcon,
  default1475 as AlarmClockMinusIcon,
  default9 as AlarmClockOffIcon,
  default1477 as AlarmClockPlusIcon,
  default1476 as AlarmMinusIcon,
  default1478 as AlarmPlusIcon,
  default11 as AlarmSmokeIcon,
  default12 as AlbumIcon,
  default1540 as AlertCircleIcon,
  default1736 as AlertOctagonIcon,
  default1922 as AlertTriangleIcon,
  default14 as AlignCenterHorizontalIcon,
  default1903 as AlignCenterIcon,
  default15 as AlignCenterVerticalIcon,
  default13 as AlignEndHorizontalIcon,
  default17 as AlignEndVerticalIcon,
  default16 as AlignHorizontalDistributeCenterIcon,
  default18 as AlignHorizontalDistributeEndIcon,
  default20 as AlignHorizontalDistributeStartIcon,
  default19 as AlignHorizontalJustifyCenterIcon,
  default21 as AlignHorizontalJustifyEndIcon,
  default22 as AlignHorizontalJustifyStartIcon,
  default23 as AlignHorizontalSpaceAroundIcon,
  default24 as AlignHorizontalSpaceBetweenIcon,
  default1907 as AlignJustifyIcon,
  default1910 as AlignLeftIcon,
  default1905 as AlignRightIcon,
  default26 as AlignStartHorizontalIcon,
  default25 as AlignStartVerticalIcon,
  default27 as AlignVerticalDistributeCenterIcon,
  default28 as AlignVerticalDistributeEndIcon,
  default29 as AlignVerticalDistributeStartIcon,
  default30 as AlignVerticalJustifyCenterIcon,
  default31 as AlignVerticalJustifyEndIcon,
  default32 as AlignVerticalJustifyStartIcon,
  default33 as AlignVerticalSpaceAroundIcon,
  default35 as AlignVerticalSpaceBetweenIcon,
  default34 as AmbulanceIcon,
  default36 as AmpersandIcon,
  default37 as AmpersandsIcon,
  default39 as AmphoraIcon,
  default38 as AnchorIcon,
  default40 as AngryIcon,
  default41 as AnnoyedIcon,
  default43 as AntennaIcon,
  default42 as AnvilIcon,
  default44 as ApertureIcon,
  default46 as AppWindowIcon,
  default45 as AppWindowMacIcon,
  default47 as AppleIcon,
  default50 as ArchiveIcon,
  default48 as ArchiveRestoreIcon,
  default49 as ArchiveXIcon,
  default1508 as AreaChartIcon,
  default51 as ArmchairIcon,
  default56 as ArrowBigDownDashIcon,
  default52 as ArrowBigDownIcon,
  default53 as ArrowBigLeftDashIcon,
  default54 as ArrowBigLeftIcon,
  default55 as ArrowBigRightDashIcon,
  default57 as ArrowBigRightIcon,
  default58 as ArrowBigUpDashIcon,
  default59 as ArrowBigUpIcon,
  default1469 as ArrowDown01Icon,
  default1470 as ArrowDown10Icon,
  default1479 as ArrowDownAZIcon,
  default1480 as ArrowDownAzIcon,
  default1536 as ArrowDownCircleIcon,
  default60 as ArrowDownFromLineIcon,
  default67 as ArrowDownIcon,
  default1544 as ArrowDownLeftFromCircleIcon,
  default1794 as ArrowDownLeftFromSquareIcon,
  default62 as ArrowDownLeftIcon,
  default1788 as ArrowDownLeftSquareIcon,
  default61 as ArrowDownNarrowWideIcon,
  default1538 as ArrowDownRightFromCircleIcon,
  default1800 as ArrowDownRightFromSquareIcon,
  default63 as ArrowDownRightIcon,
  default1790 as ArrowDownRightSquareIcon,
  default1796 as ArrowDownSquareIcon,
  default65 as ArrowDownToDotIcon,
  default64 as ArrowDownToLineIcon,
  default66 as ArrowDownUpIcon,
  default1481 as ArrowDownWideNarrowIcon,
  default1483 as ArrowDownZAIcon,
  default1484 as ArrowDownZaIcon,
  default1542 as ArrowLeftCircleIcon,
  default68 as ArrowLeftFromLineIcon,
  default71 as ArrowLeftIcon,
  default69 as ArrowLeftRightIcon,
  default1792 as ArrowLeftSquareIcon,
  default70 as ArrowLeftToLineIcon,
  default1556 as ArrowRightCircleIcon,
  default72 as ArrowRightFromLineIcon,
  default75 as ArrowRightIcon,
  default73 as ArrowRightLeftIcon,
  default1804 as ArrowRightSquareIcon,
  default74 as ArrowRightToLineIcon,
  default1471 as ArrowUp01Icon,
  default1472 as ArrowUp10Icon,
  default1485 as ArrowUpAZIcon,
  default1486 as ArrowUpAzIcon,
  default1550 as ArrowUpCircleIcon,
  default76 as ArrowUpDownIcon,
  default77 as ArrowUpFromDotIcon,
  default78 as ArrowUpFromLineIcon,
  default83 as ArrowUpIcon,
  default1546 as ArrowUpLeftFromCircleIcon,
  default1798 as ArrowUpLeftFromSquareIcon,
  default79 as ArrowUpLeftIcon,
  default1806 as ArrowUpLeftSquareIcon,
  default1487 as ArrowUpNarrowWideIcon,
  default1548 as ArrowUpRightFromCircleIcon,
  default1802 as ArrowUpRightFromSquareIcon,
  default81 as ArrowUpRightIcon,
  default1808 as ArrowUpRightSquareIcon,
  default1812 as ArrowUpSquareIcon,
  default80 as ArrowUpToLineIcon,
  default82 as ArrowUpWideNarrowIcon,
  default1489 as ArrowUpZAIcon,
  default1490 as ArrowUpZaIcon,
  default84 as ArrowsUpFromLineIcon,
  default86 as AsteriskIcon,
  default1810 as AsteriskSquareIcon,
  default85 as AtSignIcon,
  default87 as AtomIcon,
  default88 as AudioLinesIcon,
  default90 as AudioWaveformIcon,
  default89 as AwardIcon,
  default91 as AxeIcon,
  default1492 as Axis3DIcon,
  default1491 as Axis3dIcon,
  default92 as BabyIcon,
  default93 as BackpackIcon,
  default94 as BadgeAlertIcon,
  default95 as BadgeCentIcon,
  default1493 as BadgeCheckIcon,
  default96 as BadgeDollarSignIcon,
  default97 as BadgeEuroIcon,
  default1496 as BadgeHelpIcon,
  default110 as BadgeIcon,
  default98 as BadgeIndianRupeeIcon,
  default99 as BadgeInfoIcon,
  default101 as BadgeJapaneseYenIcon,
  default100 as BadgeMinusIcon,
  default102 as BadgePercentIcon,
  default103 as BadgePlusIcon,
  default104 as BadgePoundSterlingIcon,
  default1495 as BadgeQuestionMarkIcon,
  default105 as BadgeRussianRubleIcon,
  default106 as BadgeSwissFrancIcon,
  default108 as BadgeTurkishLiraIcon,
  default107 as BadgeXIcon,
  default109 as BaggageClaimIcon,
  default111 as BalloonIcon,
  default112 as BanIcon,
  default113 as BananaIcon,
  default114 as BandageIcon,
  default115 as BanknoteArrowDownIcon,
  default117 as BanknoteArrowUpIcon,
  default118 as BanknoteIcon,
  default116 as BanknoteXIcon,
  default1526 as BarChart2Icon,
  default1520 as BarChart3Icon,
  default1518 as BarChart4Icon,
  default1514 as BarChartBigIcon,
  default1510 as BarChartHorizontalBigIcon,
  default1512 as BarChartHorizontalIcon,
  default1524 as BarChartIcon,
  default120 as BarcodeIcon,
  default119 as BarrelIcon,
  default122 as BaselineIcon,
  default121 as BathIcon,
  default123 as BatteryChargingIcon,
  default124 as BatteryFullIcon,
  default130 as BatteryIcon,
  default126 as BatteryLowIcon,
  default125 as BatteryMediumIcon,
  default127 as BatteryPlusIcon,
  default128 as BatteryWarningIcon,
  default129 as BeakerIcon,
  default132 as BeanIcon,
  default131 as BeanOffIcon,
  default133 as BedDoubleIcon,
  default135 as BedIcon,
  default134 as BedSingleIcon,
  default136 as BeefIcon,
  default138 as BeerIcon,
  default137 as BeerOffIcon,
  default140 as BellDotIcon,
  default139 as BellElectricIcon,
  default145 as BellIcon,
  default141 as BellMinusIcon,
  default142 as BellOffIcon,
  default143 as BellPlusIcon,
  default144 as BellRingIcon,
  default1498 as BetweenHorizonalEndIcon,
  default1500 as BetweenHorizonalStartIcon,
  default1497 as BetweenHorizontalEndIcon,
  default1499 as BetweenHorizontalStartIcon,
  default146 as BetweenVerticalEndIcon,
  default147 as BetweenVerticalStartIcon,
  default148 as BicepsFlexedIcon,
  default149 as BikeIcon,
  default150 as BinaryIcon,
  default152 as BinocularsIcon,
  default151 as BiohazardIcon,
  default153 as BirdIcon,
  default154 as BirdhouseIcon,
  default155 as BitcoinIcon,
  default157 as BlendIcon,
  default156 as BlindsIcon,
  default158 as BlocksIcon,
  default159 as BluetoothConnectedIcon,
  default161 as BluetoothIcon,
  default160 as BluetoothOffIcon,
  default162 as BluetoothSearchingIcon,
  default163 as BoldIcon,
  default164 as BoltIcon,
  default165 as BombIcon,
  default166 as BoneIcon,
  default167 as BookAIcon,
  default168 as BookAlertIcon,
  default169 as BookAudioIcon,
  default171 as BookCheckIcon,
  default170 as BookCopyIcon,
  default1501 as BookDashedIcon,
  default172 as BookDownIcon,
  default173 as BookHeadphonesIcon,
  default174 as BookHeartIcon,
  default190 as BookIcon,
  default175 as BookImageIcon,
  default176 as BookKeyIcon,
  default179 as BookLockIcon,
  default178 as BookMarkedIcon,
  default177 as BookMinusIcon,
  default180 as BookOpenCheckIcon,
  default182 as BookOpenIcon,
  default181 as BookOpenTextIcon,
  default183 as BookPlusIcon,
  default184 as BookSearchIcon,
  default1502 as BookTemplateIcon,
  default185 as BookTextIcon,
  default186 as BookTypeIcon,
  default188 as BookUp2Icon,
  default187 as BookUpIcon,
  default189 as BookUserIcon,
  default192 as BookXIcon,
  default191 as BookmarkCheckIcon,
  default196 as BookmarkIcon,
  default193 as BookmarkMinusIcon,
  default194 as BookmarkPlusIcon,
  default195 as BookmarkXIcon,
  default197 as BoomBoxIcon,
  default200 as BotIcon,
  default199 as BotMessageSquareIcon,
  default198 as BotOffIcon,
  default201 as BottleWineIcon,
  default202 as BowArrowIcon,
  default203 as BoxIcon,
  default1841 as BoxSelectIcon,
  default204 as BoxesIcon,
  default1503 as BracesIcon,
  default205 as BracketsIcon,
  default207 as BrainCircuitIcon,
  default206 as BrainCogIcon,
  default208 as BrainIcon,
  default210 as BrickWallFireIcon,
  default211 as BrickWallIcon,
  default209 as BrickWallShieldIcon,
  default213 as BriefcaseBusinessIcon,
  default212 as BriefcaseConveyorBeltIcon,
  default215 as BriefcaseIcon,
  default214 as BriefcaseMedicalIcon,
  default216 as BringToFrontIcon,
  default217 as BrushCleaningIcon,
  default218 as BrushIcon,
  default219 as BubblesIcon,
  default222 as BugIcon,
  default220 as BugOffIcon,
  default221 as BugPlayIcon,
  default223 as Building2Icon,
  default225 as BuildingIcon,
  default224 as BusFrontIcon,
  default226 as BusIcon,
  default227 as CableCarIcon,
  default229 as CableIcon,
  default230 as CakeIcon,
  default228 as CakeSliceIcon,
  default231 as CalculatorIcon,
  default232 as Calendar1Icon,
  default235 as CalendarArrowDownIcon,
  default233 as CalendarArrowUpIcon,
  default234 as CalendarCheck2Icon,
  default236 as CalendarCheckIcon,
  default237 as CalendarClockIcon,
  default239 as CalendarCogIcon,
  default238 as CalendarDaysIcon,
  default240 as CalendarFoldIcon,
  default241 as CalendarHeartIcon,
  default251 as CalendarIcon,
  default243 as CalendarMinus2Icon,
  default242 as CalendarMinusIcon,
  default244 as CalendarOffIcon,
  default245 as CalendarPlus2Icon,
  default246 as CalendarPlusIcon,
  default247 as CalendarRangeIcon,
  default248 as CalendarSearchIcon,
  default249 as CalendarSyncIcon,
  default250 as CalendarX2Icon,
  default252 as CalendarXIcon,
  default253 as CalendarsIcon,
  default255 as CameraIcon,
  default254 as CameraOffIcon,
  default1516 as CandlestickChartIcon,
  default256 as CandyCaneIcon,
  default257 as CandyIcon,
  default258 as CandyOffIcon,
  default260 as CannabisIcon,
  default259 as CannabisOffIcon,
  default1505 as CaptionsIcon,
  default261 as CaptionsOffIcon,
  default262 as CarFrontIcon,
  default263 as CarIcon,
  default264 as CarTaxiFrontIcon,
  default265 as CaravanIcon,
  default266 as CardSimIcon,
  default267 as CarrotIcon,
  default268 as CaseLowerIcon,
  default269 as CaseSensitiveIcon,
  default270 as CaseUpperIcon,
  default271 as CassetteTapeIcon,
  default272 as CastIcon,
  default273 as CastleIcon,
  default274 as CatIcon,
  default276 as CctvIcon,
  default1507 as ChartAreaIcon,
  default1509 as ChartBarBigIcon,
  default275 as ChartBarDecreasingIcon,
  default1511 as ChartBarIcon,
  default277 as ChartBarIncreasingIcon,
  default278 as ChartBarStackedIcon,
  default1515 as ChartCandlestickIcon,
  default1513 as ChartColumnBigIcon,
  default280 as ChartColumnDecreasingIcon,
  default1519 as ChartColumnIcon,
  default1517 as ChartColumnIncreasingIcon,
  default279 as ChartColumnStackedIcon,
  default281 as ChartGanttIcon,
  default1521 as ChartLineIcon,
  default282 as ChartNetworkIcon,
  default283 as ChartNoAxesColumnDecreasingIcon,
  default1525 as ChartNoAxesColumnIcon,
  default1523 as ChartNoAxesColumnIncreasingIcon,
  default284 as ChartNoAxesCombinedIcon,
  default1527 as ChartNoAxesGanttIcon,
  default1529 as ChartPieIcon,
  default1531 as ChartScatterIcon,
  default286 as ChartSplineIcon,
  default285 as CheckCheckIcon,
  default1554 as CheckCircle2Icon,
  default1552 as CheckCircleIcon,
  default288 as CheckIcon,
  default287 as CheckLineIcon,
  default1825 as CheckSquare2Icon,
  default1820 as CheckSquareIcon,
  default290 as ChefHatIcon,
  default289 as CherryIcon,
  default292 as ChessBishopIcon,
  default291 as ChessKingIcon,
  default293 as ChessKnightIcon,
  default294 as ChessPawnIcon,
  default295 as ChessQueenIcon,
  default296 as ChessRookIcon,
  default1558 as ChevronDownCircleIcon,
  default297 as ChevronDownIcon,
  default1829 as ChevronDownSquareIcon,
  default298 as ChevronFirstIcon,
  default299 as ChevronLastIcon,
  default1560 as ChevronLeftCircleIcon,
  default300 as ChevronLeftIcon,
  default1827 as ChevronLeftSquareIcon,
  default1562 as ChevronRightCircleIcon,
  default301 as ChevronRightIcon,
  default1831 as ChevronRightSquareIcon,
  default1564 as ChevronUpCircleIcon,
  default302 as ChevronUpIcon,
  default1833 as ChevronUpSquareIcon,
  default304 as ChevronsDownIcon,
  default303 as ChevronsDownUpIcon,
  default307 as ChevronsLeftIcon,
  default305 as ChevronsLeftRightEllipsisIcon,
  default306 as ChevronsLeftRightIcon,
  default309 as ChevronsRightIcon,
  default308 as ChevronsRightLeftIcon,
  default310 as ChevronsUpDownIcon,
  default311 as ChevronsUpIcon,
  default1534 as ChromeIcon,
  default1533 as ChromiumIcon,
  default312 as ChurchIcon,
  default314 as CigaretteIcon,
  default313 as CigaretteOffIcon,
  default1539 as CircleAlertIcon,
  default1535 as CircleArrowDownIcon,
  default1541 as CircleArrowLeftIcon,
  default1543 as CircleArrowOutDownLeftIcon,
  default1537 as CircleArrowOutDownRightIcon,
  default1545 as CircleArrowOutUpLeftIcon,
  default1547 as CircleArrowOutUpRightIcon,
  default1555 as CircleArrowRightIcon,
  default1549 as CircleArrowUpIcon,
  default1551 as CircleCheckBigIcon,
  default1553 as CircleCheckIcon,
  default1557 as CircleChevronDownIcon,
  default1559 as CircleChevronLeftIcon,
  default1561 as CircleChevronRightIcon,
  default1563 as CircleChevronUpIcon,
  default315 as CircleDashedIcon,
  default1565 as CircleDivideIcon,
  default316 as CircleDollarSignIcon,
  default317 as CircleDotDashedIcon,
  default319 as CircleDotIcon,
  default318 as CircleEllipsisIcon,
  default320 as CircleEqualIcon,
  default321 as CircleFadingArrowUpIcon,
  default322 as CircleFadingPlusIcon,
  default1567 as CircleGaugeIcon,
  default1585 as CircleHelpIcon,
  default329 as CircleIcon,
  default1569 as CircleMinusIcon,
  default323 as CircleOffIcon,
  default1575 as CircleParkingIcon,
  default1571 as CircleParkingOffIcon,
  default1573 as CirclePauseIcon,
  default1577 as CirclePercentIcon,
  default324 as CirclePileIcon,
  default1579 as CirclePlayIcon,
  default1581 as CirclePlusIcon,
  default325 as CirclePoundSterlingIcon,
  default1586 as CirclePowerIcon,
  default1583 as CircleQuestionMarkIcon,
  default1588 as CircleSlash2Icon,
  default326 as CircleSlashIcon,
  default1589 as CircleSlashedIcon,
  default327 as CircleSmallIcon,
  default328 as CircleStarIcon,
  default1590 as CircleStopIcon,
  default1592 as CircleUserIcon,
  default1594 as CircleUserRoundIcon,
  default1596 as CircleXIcon,
  default331 as CircuitBoardIcon,
  default330 as CitrusIcon,
  default332 as ClapperboardIcon,
  default333 as ClipboardCheckIcon,
  default334 as ClipboardClockIcon,
  default335 as ClipboardCopyIcon,
  default1599 as ClipboardEditIcon,
  default342 as ClipboardIcon,
  default336 as ClipboardListIcon,
  default337 as ClipboardMinusIcon,
  default338 as ClipboardPasteIcon,
  default1598 as ClipboardPenIcon,
  default1600 as ClipboardPenLineIcon,
  default339 as ClipboardPlusIcon,
  default1601 as ClipboardSignatureIcon,
  default340 as ClipboardTypeIcon,
  default341 as ClipboardXIcon,
  default343 as Clock10Icon,
  default345 as Clock11Icon,
  default346 as Clock12Icon,
  default344 as Clock1Icon,
  default347 as Clock2Icon,
  default349 as Clock3Icon,
  default348 as Clock4Icon,
  default350 as Clock5Icon,
  default351 as Clock6Icon,
  default352 as Clock7Icon,
  default353 as Clock8Icon,
  default355 as Clock9Icon,
  default354 as ClockAlertIcon,
  default356 as ClockArrowDownIcon,
  default357 as ClockArrowUpIcon,
  default358 as ClockCheckIcon,
  default359 as ClockFadingIcon,
  default361 as ClockIcon,
  default360 as ClockPlusIcon,
  default362 as ClosedCaptionIcon,
  default363 as CloudAlertIcon,
  default364 as CloudBackupIcon,
  default366 as CloudCheckIcon,
  default365 as CloudCogIcon,
  default1602 as CloudDownloadIcon,
  default367 as CloudDrizzleIcon,
  default368 as CloudFogIcon,
  default370 as CloudHailIcon,
  default381 as CloudIcon,
  default369 as CloudLightningIcon,
  default373 as CloudMoonIcon,
  default371 as CloudMoonRainIcon,
  default372 as CloudOffIcon,
  default374 as CloudRainIcon,
  default375 as CloudRainWindIcon,
  default376 as CloudSnowIcon,
  default378 as CloudSunIcon,
  default377 as CloudSunRainIcon,
  default379 as CloudSyncIcon,
  default1604 as CloudUploadIcon,
  default380 as CloudyIcon,
  default382 as CloverIcon,
  default383 as ClubIcon,
  default1607 as Code2Icon,
  default385 as CodeIcon,
  default1835 as CodeSquareIcon,
  default1606 as CodeXmlIcon,
  default384 as CodepenIcon,
  default386 as CodesandboxIcon,
  default387 as CoffeeIcon,
  default388 as CogIcon,
  default389 as CoinsIcon,
  default1608 as Columns2Icon,
  default1610 as Columns3CogIcon,
  default1613 as Columns3Icon,
  default390 as Columns4Icon,
  default1609 as ColumnsIcon,
  default1611 as ColumnsSettingsIcon,
  default391 as CombineIcon,
  default392 as CommandIcon,
  default394 as CompassIcon,
  default393 as ComponentIcon,
  default395 as ComputerIcon,
  default396 as ConciergeBellIcon,
  default397 as ConeIcon,
  default398 as ConstructionIcon,
  default1616 as Contact2Icon,
  default399 as ContactIcon,
  default1615 as ContactRoundIcon,
  default400 as ContainerIcon,
  default401 as ContrastIcon,
  default402 as CookieIcon,
  default404 as CookingPotIcon,
  default403 as CopyCheckIcon,
  default409 as CopyIcon,
  default405 as CopyMinusIcon,
  default406 as CopyPlusIcon,
  default407 as CopySlashIcon,
  default408 as CopyXIcon,
  default410 as CopyleftIcon,
  default411 as CopyrightIcon,
  default412 as CornerDownLeftIcon,
  default413 as CornerDownRightIcon,
  default414 as CornerLeftDownIcon,
  default415 as CornerLeftUpIcon,
  default416 as CornerRightDownIcon,
  default417 as CornerRightUpIcon,
  default418 as CornerUpLeftIcon,
  default419 as CornerUpRightIcon,
  default420 as CpuIcon,
  default421 as CreativeCommonsIcon,
  default422 as CreditCardIcon,
  default424 as CroissantIcon,
  default423 as CropIcon,
  default426 as CrossIcon,
  default425 as CrosshairIcon,
  default428 as CrownIcon,
  default427 as CuboidIcon,
  default429 as CupSodaIcon,
  default1504 as CurlyBracesIcon,
  default430 as CurrencyIcon,
  default431 as CylinderIcon,
  default432 as DamIcon,
  default433 as DatabaseBackupIcon,
  default437 as DatabaseIcon,
  default434 as DatabaseSearchIcon,
  default435 as DatabaseZapIcon,
  default436 as DecimalsArrowLeftIcon,
  default438 as DecimalsArrowRightIcon,
  default439 as DeleteIcon,
  default441 as DessertIcon,
  default440 as DiameterIcon,
  default443 as DiamondIcon,
  default442 as DiamondMinusIcon,
  default1617 as DiamondPercentIcon,
  default444 as DiamondPlusIcon,
  default445 as Dice1Icon,
  default446 as Dice2Icon,
  default447 as Dice3Icon,
  default448 as Dice4Icon,
  default449 as Dice5Icon,
  default451 as Dice6Icon,
  default450 as DicesIcon,
  default452 as DiffIcon,
  default453 as Disc2Icon,
  default454 as Disc3Icon,
  default455 as DiscAlbumIcon,
  default457 as DiscIcon,
  default1566 as DivideCircleIcon,
  default456 as DivideIcon,
  default1843 as DivideSquareIcon,
  default459 as DnaIcon,
  default458 as DnaOffIcon,
  default460 as DockIcon,
  default461 as DogIcon,
  default463 as DollarSignIcon,
  default462 as DonutIcon,
  default466 as DoorClosedIcon,
  default464 as DoorClosedLockedIcon,
  default465 as DoorOpenIcon,
  default467 as DotIcon,
  default1845 as DotSquareIcon,
  default1603 as DownloadCloudIcon,
  default468 as DownloadIcon,
  default469 as DraftingCompassIcon,
  default470 as DramaIcon,
  default471 as DribbbleIcon,
  default472 as DrillIcon,
  default473 as DroneIcon,
  default474 as DropletIcon,
  default477 as DropletOffIcon,
  default475 as DropletsIcon,
  default476 as DrumIcon,
  default478 as DrumstickIcon,
  default479 as DumbbellIcon,
  default480 as EarIcon,
  default482 as EarOffIcon,
  default1619 as EarthIcon,
  default481 as EarthLockIcon,
  default483 as EclipseIcon,
  default1762 as Edit2Icon,
  default1760 as Edit3Icon,
  default1868 as EditIcon,
  default484 as EggFriedIcon,
  default486 as EggIcon,
  default485 as EggOffIcon,
  default487 as EllipseIcon,
  default1623 as EllipsisIcon,
  default1621 as EllipsisVerticalIcon,
  default488 as EqualApproximatelyIcon,
  default490 as EqualIcon,
  default489 as EqualNotIcon,
  default1847 as EqualSquareIcon,
  default491 as EraserIcon,
  default492 as EthernetPortIcon,
  default493 as EuroIcon,
  default494 as EvChargerIcon,
  default495 as ExpandIcon,
  default496 as ExternalLinkIcon,
  default497 as EyeClosedIcon,
  default499 as EyeIcon,
  default498 as EyeOffIcon,
  default500 as FacebookIcon,
  default501 as FactoryIcon,
  default502 as FanIcon,
  default503 as FastForwardIcon,
  default504 as FeatherIcon,
  default505 as FenceIcon,
  default506 as FerrisWheelIcon,
  default507 as FigmaIcon,
  default508 as FileArchiveIcon,
  default1651 as FileAudio2Icon,
  default1650 as FileAudioIcon,
  default1626 as FileAxis3DIcon,
  default1625 as FileAxis3dIcon,
  default1632 as FileBadge2Icon,
  default1631 as FileBadgeIcon,
  default1634 as FileBarChart2Icon,
  default1636 as FileBarChartIcon,
  default509 as FileBoxIcon,
  default1629 as FileBracesCornerIcon,
  default1627 as FileBracesIcon,
  default1633 as FileChartColumnIcon,
  default1635 as FileChartColumnIncreasingIcon,
  default1637 as FileChartLineIcon,
  default1639 as FileChartPieIcon,
  default1642 as FileCheck2Icon,
  default1641 as FileCheckCornerIcon,
  default510 as FileCheckIcon,
  default511 as FileClockIcon,
  default1644 as FileCode2Icon,
  default1643 as FileCodeCornerIcon,
  default512 as FileCodeIcon,
  default1648 as FileCog2Icon,
  default1647 as FileCogIcon,
  default513 as FileDiffIcon,
  default514 as FileDigitIcon,
  default515 as FileDownIcon,
  default1661 as FileEditIcon,
  default1645 as FileExclamationPointIcon,
  default1649 as FileHeadphoneIcon,
  default517 as FileHeartIcon,
  default536 as FileIcon,
  default516 as FileImageIcon,
  default518 as FileInputIcon,
  default1630 as FileJson2Icon,
  default1628 as FileJsonIcon,
  default1655 as FileKey2Icon,
  default1654 as FileKeyIcon,
  default1638 as FileLineChartIcon,
  default1653 as FileLock2Icon,
  default1652 as FileLockIcon,
  default1657 as FileMinus2Icon,
  default1656 as FileMinusCornerIcon,
  default519 as FileMinusIcon,
  default520 as FileMusicIcon,
  default521 as FileOutputIcon,
  default1660 as FilePenIcon,
  default1658 as FilePenLineIcon,
  default1640 as FilePieChartIcon,
  default1662 as FilePlayIcon,
  default1665 as FilePlus2Icon,
  default1664 as FilePlusCornerIcon,
  default522 as FilePlusIcon,
  default1667 as FileQuestionIcon,
  default1666 as FileQuestionMarkIcon,
  default523 as FileScanIcon,
  default1669 as FileSearch2Icon,
  default1668 as FileSearchCornerIcon,
  default524 as FileSearchIcon,
  default1670 as FileSignalIcon,
  default1659 as FileSignatureIcon,
  default525 as FileSlidersIcon,
  default526 as FileSpreadsheetIcon,
  default527 as FileStackIcon,
  default528 as FileSymlinkIcon,
  default529 as FileTerminalIcon,
  default531 as FileTextIcon,
  default1673 as FileType2Icon,
  default1672 as FileTypeCornerIcon,
  default530 as FileTypeIcon,
  default533 as FileUpIcon,
  default532 as FileUserIcon,
  default1675 as FileVideo2Icon,
  default1674 as FileVideoCameraIcon,
  default1663 as FileVideoIcon,
  default1671 as FileVolume2Icon,
  default534 as FileVolumeIcon,
  default1646 as FileWarningIcon,
  default1677 as FileX2Icon,
  default1676 as FileXCornerIcon,
  default535 as FileXIcon,
  default537 as FilesIcon,
  default538 as FilmIcon,
  default1685 as FilterIcon,
  default1683 as FilterXIcon,
  default1468 as FingerprintIcon,
  default1467 as FingerprintPatternIcon,
  default539 as FireExtinguisherIcon,
  default541 as FishIcon,
  default540 as FishOffIcon,
  default542 as FishSymbolIcon,
  default543 as FishingHookIcon,
  default544 as FishingRodIcon,
  default548 as FlagIcon,
  default545 as FlagOffIcon,
  default546 as FlagTriangleLeftIcon,
  default547 as FlagTriangleRightIcon,
  default551 as FlameIcon,
  default549 as FlameKindlingIcon,
  default553 as FlashlightIcon,
  default550 as FlashlightOffIcon,
  default554 as FlaskConicalIcon,
  default552 as FlaskConicalOffIcon,
  default555 as FlaskRoundIcon,
  default556 as FlipHorizontal2Icon,
  default1814 as FlipHorizontalIcon,
  default557 as FlipVertical2Icon,
  default1818 as FlipVerticalIcon,
  default558 as Flower2Icon,
  default559 as FlowerIcon,
  default561 as FocusIcon,
  default560 as FoldHorizontalIcon,
  default562 as FoldVerticalIcon,
  default563 as FolderArchiveIcon,
  default564 as FolderCheckIcon,
  default565 as FolderClockIcon,
  default566 as FolderClosedIcon,
  default567 as FolderCodeIcon,
  default1679 as FolderCog2Icon,
  default1678 as FolderCogIcon,
  default568 as FolderDotIcon,
  default569 as FolderDownIcon,
  default1681 as FolderEditIcon,
  default570 as FolderGit2Icon,
  default571 as FolderGitIcon,
  default572 as FolderHeartIcon,
  default592 as FolderIcon,
  default573 as FolderInputIcon,
  default575 as FolderKanbanIcon,
  default574 as FolderKeyIcon,
  default576 as FolderLockIcon,
  default577 as FolderMinusIcon,
  default578 as FolderOpenDotIcon,
  default579 as FolderOpenIcon,
  default580 as FolderOutputIcon,
  default1680 as FolderPenIcon,
  default581 as FolderPlusIcon,
  default582 as FolderRootIcon,
  default583 as FolderSearch2Icon,
  default584 as FolderSearchIcon,
  default586 as FolderSymlinkIcon,
  default585 as FolderSyncIcon,
  default587 as FolderTreeIcon,
  default588 as FolderUpIcon,
  default589 as FolderXIcon,
  default590 as FoldersIcon,
  default591 as FootprintsIcon,
  default1944 as ForkKnifeCrossedIcon,
  default1942 as ForkKnifeIcon,
  default593 as ForkliftIcon,
  default594 as FormIcon,
  default1766 as FormInputIcon,
  default595 as ForwardIcon,
  default596 as FrameIcon,
  default598 as FramerIcon,
  default597 as FrownIcon,
  default599 as FuelIcon,
  default600 as FullscreenIcon,
  default1849 as FunctionSquareIcon,
  default1684 as FunnelIcon,
  default601 as FunnelPlusIcon,
  default1682 as FunnelXIcon,
  default604 as GalleryHorizontalEndIcon,
  default602 as GalleryHorizontalIcon,
  default603 as GalleryThumbnailsIcon,
  default606 as GalleryVerticalEndIcon,
  default605 as GalleryVerticalIcon,
  default607 as Gamepad2Icon,
  default608 as GamepadDirectionalIcon,
  default609 as GamepadIcon,
  default1528 as GanttChartIcon,
  default1822 as GanttChartSquareIcon,
  default1568 as GaugeCircleIcon,
  default610 as GaugeIcon,
  default611 as GavelIcon,
  default612 as GemIcon,
  default613 as GeorgianLariIcon,
  default614 as GhostIcon,
  default617 as GiftIcon,
  default616 as GitBranchIcon,
  default615 as GitBranchMinusIcon,
  default618 as GitBranchPlusIcon,
  default1686 as GitCommitHorizontalIcon,
  default1687 as GitCommitIcon,
  default619 as GitCommitVerticalIcon,
  default620 as GitCompareArrowsIcon,
  default621 as GitCompareIcon,
  default622 as GitForkIcon,
  default623 as GitGraphIcon,
  default624 as GitMergeConflictIcon,
  default625 as GitMergeIcon,
  default626 as GitPullRequestArrowIcon,
  default627 as GitPullRequestClosedIcon,
  default628 as GitPullRequestCreateArrowIcon,
  default630 as GitPullRequestCreateIcon,
  default629 as GitPullRequestDraftIcon,
  default631 as GitPullRequestIcon,
  default632 as GithubIcon,
  default633 as GitlabIcon,
  default634 as GlassWaterIcon,
  default635 as GlassesIcon,
  default1620 as Globe2Icon,
  default641 as GlobeIcon,
  default636 as GlobeLockIcon,
  default638 as GlobeOffIcon,
  default637 as GlobeXIcon,
  default639 as GoalIcon,
  default640 as GpuIcon,
  default1700 as GrabIcon,
  default642 as GraduationCapIcon,
  default643 as GrapeIcon,
  default1691 as Grid2X2CheckIcon,
  default1693 as Grid2X2Icon,
  default1689 as Grid2X2PlusIcon,
  default1695 as Grid2X2XIcon,
  default1690 as Grid2x2CheckIcon,
  default1692 as Grid2x2Icon,
  default1688 as Grid2x2PlusIcon,
  default1694 as Grid2x2XIcon,
  default1698 as Grid3X3Icon,
  default644 as Grid3x2Icon,
  default1696 as Grid3x3Icon,
  default1697 as GridIcon,
  default645 as GripHorizontalIcon,
  default647 as GripIcon,
  default646 as GripVerticalIcon,
  default648 as GroupIcon,
  default652 as GuitarIcon,
  default649 as HamIcon,
  default650 as HamburgerIcon,
  default651 as HammerIcon,
  default653 as HandCoinsIcon,
  default654 as HandFistIcon,
  default1699 as HandGrabIcon,
  default655 as HandHeartIcon,
  default1701 as HandHelpingIcon,
  default657 as HandIcon,
  default656 as HandMetalIcon,
  default658 as HandPlatterIcon,
  default659 as HandbagIcon,
  default660 as HandshakeIcon,
  default662 as HardDriveDownloadIcon,
  default663 as HardDriveIcon,
  default661 as HardDriveUploadIcon,
  default664 as HardHatIcon,
  default665 as HashIcon,
  default666 as HatGlassesIcon,
  default667 as HazeIcon,
  default668 as HdIcon,
  default669 as HdmiPortIcon,
  default670 as Heading1Icon,
  default671 as Heading2Icon,
  default672 as Heading3Icon,
  default673 as Heading4Icon,
  default675 as Heading5Icon,
  default674 as Heading6Icon,
  default676 as HeadingIcon,
  default677 as HeadphoneOffIcon,
  default678 as HeadphonesIcon,
  default679 as HeadsetIcon,
  default680 as HeartCrackIcon,
  default681 as HeartHandshakeIcon,
  default686 as HeartIcon,
  default682 as HeartMinusIcon,
  default683 as HeartOffIcon,
  default684 as HeartPlusIcon,
  default685 as HeartPulseIcon,
  default687 as HeaterIcon,
  default688 as HelicopterIcon,
  default1584 as HelpCircleIcon,
  default1702 as HelpingHandIcon,
  default689 as HexagonIcon,
  default690 as HighlighterIcon,
  default691 as HistoryIcon,
  default1704 as HomeIcon,
  default693 as HopIcon,
  default692 as HopOffIcon,
  default695 as HospitalIcon,
  default694 as HotelIcon,
  default696 as HourglassIcon,
  default697 as HouseHeartIcon,
  default1703 as HouseIcon,
  default698 as HousePlugIcon,
  default700 as HousePlusIcon,
  default699 as HouseWifiIcon,
  default1706 as IceCream2Icon,
  default1705 as IceCreamBowlIcon,
  default1707 as IceCreamConeIcon,
  default1708 as IceCreamIcon,
  default703 as IdCardIcon,
  default701 as IdCardLanyardIcon,
  default702 as ImageDownIcon,
  default710 as ImageIcon,
  default704 as ImageMinusIcon,
  default706 as ImageOffIcon,
  default705 as ImagePlayIcon,
  default707 as ImagePlusIcon,
  default708 as ImageUpIcon,
  default709 as ImageUpscaleIcon,
  default712 as ImagesIcon,
  default711 as ImportIcon,
  default714 as InboxIcon,
  default1715 as IndentDecreaseIcon,
  default1717 as IndentIcon,
  default1718 as IndentIncreaseIcon,
  default713 as IndianRupeeIcon,
  default715 as InfinityIcon,
  default716 as InfoIcon,
  default1859 as InspectIcon,
  default717 as InspectionPanelIcon,
  default718 as InstagramIcon,
  default719 as ItalicIcon,
  default720 as IterationCcwIcon,
  default721 as IterationCwIcon,
  default722 as JapaneseYenIcon,
  default723 as JoystickIcon,
  default724 as KanbanIcon,
  default1837 as KanbanSquareDashedIcon,
  default1851 as KanbanSquareIcon,
  default725 as KayakIcon,
  default728 as KeyIcon,
  default726 as KeyRoundIcon,
  default727 as KeySquareIcon,
  default730 as KeyboardIcon,
  default729 as KeyboardMusicIcon,
  default731 as KeyboardOffIcon,
  default734 as LampCeilingIcon,
  default732 as LampDeskIcon,
  default733 as LampFloorIcon,
  default737 as LampIcon,
  default735 as LampWallDownIcon,
  default736 as LampWallUpIcon,
  default738 as LandPlotIcon,
  default739 as LandmarkIcon,
  default740 as LanguagesIcon,
  default1710 as Laptop2Icon,
  default742 as LaptopIcon,
  default741 as LaptopMinimalCheckIcon,
  default1709 as LaptopMinimalIcon,
  default745 as LassoIcon,
  default743 as LassoSelectIcon,
  default744 as LaughIcon,
  default746 as Layers2Icon,
  default1712 as Layers3Icon,
  default1711 as LayersIcon,
  default747 as LayersPlusIcon,
  default748 as LayoutDashboardIcon,
  default750 as LayoutGridIcon,
  default1758 as LayoutIcon,
  default749 as LayoutListIcon,
  default751 as LayoutPanelLeftIcon,
  default753 as LayoutPanelTopIcon,
  default752 as LayoutTemplateIcon,
  default755 as LeafIcon,
  default758 as LeafyGreenIcon,
  default754 as LecternIcon,
  default756 as LensConcaveIcon,
  default757 as LensConvexIcon,
  default1912 as LetterTextIcon,
  default759 as LibraryBigIcon,
  default760 as LibraryIcon,
  default1853 as LibrarySquareIcon,
  default761 as LifeBuoyIcon,
  default762 as LigatureIcon,
  default764 as LightbulbIcon,
  default763 as LightbulbOffIcon,
  default1522 as LineChartIcon,
  default765 as LineDotRightHorizontalIcon,
  default766 as LineSquiggleIcon,
  default768 as Link2Icon,
  default767 as Link2OffIcon,
  default769 as LinkIcon,
  default771 as LinkedinIcon,
  default770 as ListCheckIcon,
  default772 as ListChecksIcon,
  default773 as ListChevronsDownUpIcon,
  default774 as ListChevronsUpDownIcon,
  default775 as ListCollapseIcon,
  default776 as ListEndIcon,
  default778 as ListFilterIcon,
  default777 as ListFilterPlusIcon,
  default790 as ListIcon,
  default1713 as ListIndentDecreaseIcon,
  default1716 as ListIndentIncreaseIcon,
  default779 as ListMinusIcon,
  default780 as ListMusicIcon,
  default781 as ListOrderedIcon,
  default782 as ListPlusIcon,
  default783 as ListRestartIcon,
  default784 as ListStartIcon,
  default785 as ListTodoIcon,
  default786 as ListTreeIcon,
  default787 as ListVideoIcon,
  default788 as ListXIcon,
  default1720 as Loader2Icon,
  default1719 as LoaderCircleIcon,
  default791 as LoaderIcon,
  default789 as LoaderPinwheelIcon,
  default792 as LocateFixedIcon,
  default794 as LocateIcon,
  default793 as LocateOffIcon,
  default1728 as LocationEditIcon,
  default796 as LockIcon,
  default795 as LockKeyholeIcon,
  default1721 as LockKeyholeOpenIcon,
  default1725 as LockOpenIcon,
  default797 as LogInIcon,
  default798 as LogOutIcon,
  default799 as LogsIcon,
  default800 as LollipopIcon,
  default801 as LuggageIcon,
  default1855 as MSquareIcon,
  default803 as MagnetIcon,
  default802 as MailCheckIcon,
  default810 as MailIcon,
  default804 as MailMinusIcon,
  default805 as MailOpenIcon,
  default807 as MailPlusIcon,
  default1724 as MailQuestionIcon,
  default1723 as MailQuestionMarkIcon,
  default806 as MailSearchIcon,
  default808 as MailWarningIcon,
  default809 as MailXIcon,
  default811 as MailboxIcon,
  default812 as MailsIcon,
  default827 as MapIcon,
  default814 as MapMinusIcon,
  default815 as MapPinCheckIcon,
  default813 as MapPinCheckInsideIcon,
  default816 as MapPinHouseIcon,
  default824 as MapPinIcon,
  default818 as MapPinMinusIcon,
  default817 as MapPinMinusInsideIcon,
  default819 as MapPinOffIcon,
  default1727 as MapPinPenIcon,
  default820 as MapPinPlusIcon,
  default821 as MapPinPlusInsideIcon,
  default823 as MapPinXIcon,
  default822 as MapPinXInsideIcon,
  default825 as MapPinnedIcon,
  default826 as MapPlusIcon,
  default829 as MarsIcon,
  default828 as MarsStrokeIcon,
  default830 as MartiniIcon,
  default831 as Maximize2Icon,
  default832 as MaximizeIcon,
  default833 as MedalIcon,
  default835 as MegaphoneIcon,
  default834 as MegaphoneOffIcon,
  default836 as MehIcon,
  default837 as MemoryStickIcon,
  default838 as MenuIcon,
  default1857 as MenuSquareIcon,
  default839 as MergeIcon,
  default840 as MessageCircleCheckIcon,
  default841 as MessageCircleCodeIcon,
  default842 as MessageCircleDashedIcon,
  default843 as MessageCircleHeartIcon,
  default850 as MessageCircleIcon,
  default844 as MessageCircleMoreIcon,
  default846 as MessageCircleOffIcon,
  default845 as MessageCirclePlusIcon,
  default1730 as MessageCircleQuestionIcon,
  default1729 as MessageCircleQuestionMarkIcon,
  default847 as MessageCircleReplyIcon,
  default848 as MessageCircleWarningIcon,
  default849 as MessageCircleXIcon,
  default851 as MessageSquareCheckIcon,
  default854 as MessageSquareCodeIcon,
  default852 as MessageSquareDashedIcon,
  default853 as MessageSquareDiffIcon,
  default855 as MessageSquareDotIcon,
  default856 as MessageSquareHeartIcon,
  default866 as MessageSquareIcon,
  default858 as MessageSquareLockIcon,
  default857 as MessageSquareMoreIcon,
  default859 as MessageSquareOffIcon,
  default860 as MessageSquarePlusIcon,
  default861 as MessageSquareQuoteIcon,
  default862 as MessageSquareReplyIcon,
  default863 as MessageSquareShareIcon,
  default864 as MessageSquareTextIcon,
  default865 as MessageSquareWarningIcon,
  default867 as MessageSquareXIcon,
  default868 as MessagesSquareIcon,
  default869 as MetronomeIcon,
  default1732 as Mic2Icon,
  default872 as MicIcon,
  default870 as MicOffIcon,
  default1731 as MicVocalIcon,
  default871 as MicrochipIcon,
  default874 as MicroscopeIcon,
  default873 as MicrowaveIcon,
  default875 as MilestoneIcon,
  default878 as MilkIcon,
  default876 as MilkOffIcon,
  default877 as Minimize2Icon,
  default880 as MinimizeIcon,
  default1570 as MinusCircleIcon,
  default879 as MinusIcon,
  default1863 as MinusSquareIcon,
  default881 as MirrorRectangularIcon,
  default882 as MirrorRoundIcon,
  default883 as MonitorCheckIcon,
  default884 as MonitorCloudIcon,
  default885 as MonitorCogIcon,
  default886 as MonitorDotIcon,
  default888 as MonitorDownIcon,
  default896 as MonitorIcon,
  default887 as MonitorOffIcon,
  default892 as MonitorPauseIcon,
  default889 as MonitorPlayIcon,
  default890 as MonitorSmartphoneIcon,
  default891 as MonitorSpeakerIcon,
  default893 as MonitorStopIcon,
  default894 as MonitorUpIcon,
  default895 as MonitorXIcon,
  default898 as MoonIcon,
  default897 as MoonStarIcon,
  default1624 as MoreHorizontalIcon,
  default1622 as MoreVerticalIcon,
  default899 as MotorbikeIcon,
  default901 as MountainIcon,
  default900 as MountainSnowIcon,
  default911 as MouseIcon,
  default902 as MouseLeftIcon,
  default903 as MouseOffIcon,
  default905 as MousePointer2Icon,
  default904 as MousePointer2OffIcon,
  default906 as MousePointerBanIcon,
  default907 as MousePointerClickIcon,
  default909 as MousePointerIcon,
  default1839 as MousePointerSquareDashedIcon,
  default908 as MouseRightIcon,
  default1734 as Move3DIcon,
  default1733 as Move3dIcon,
  default910 as MoveDiagonal2Icon,
  default912 as MoveDiagonalIcon,
  default915 as MoveDownIcon,
  default913 as MoveDownLeftIcon,
  default914 as MoveDownRightIcon,
  default917 as MoveHorizontalIcon,
  default923 as MoveIcon,
  default916 as MoveLeftIcon,
  default919 as MoveRightIcon,
  default921 as MoveUpIcon,
  default918 as MoveUpLeftIcon,
  default920 as MoveUpRightIcon,
  default922 as MoveVerticalIcon,
  default924 as Music2Icon,
  default925 as Music3Icon,
  default926 as Music4Icon,
  default927 as MusicIcon,
  default929 as Navigation2Icon,
  default928 as Navigation2OffIcon,
  default931 as NavigationIcon,
  default930 as NavigationOffIcon,
  default932 as NetworkIcon,
  default933 as NewspaperIcon,
  default934 as NfcIcon,
  default935 as NonBinaryIcon,
  default939 as NotebookIcon,
  default936 as NotebookPenIcon,
  default937 as NotebookTabsIcon,
  default938 as NotebookTextIcon,
  default940 as NotepadTextDashedIcon,
  default941 as NotepadTextIcon,
  default943 as NutIcon,
  default942 as NutOffIcon,
  default1735 as OctagonAlertIcon,
  default945 as OctagonIcon,
  default944 as OctagonMinusIcon,
  default1737 as OctagonPauseIcon,
  default1739 as OctagonXIcon,
  default946 as OmegaIcon,
  default947 as OptionIcon,
  default949 as OrbitIcon,
  default948 as OrigamiIcon,
  default1714 as OutdentIcon,
  default950 as Package2Icon,
  default951 as PackageCheckIcon,
  default957 as PackageIcon,
  default953 as PackageMinusIcon,
  default952 as PackageOpenIcon,
  default954 as PackagePlusIcon,
  default955 as PackageSearchIcon,
  default956 as PackageXIcon,
  default958 as PaintBucketIcon,
  default959 as PaintRollerIcon,
  default1742 as Paintbrush2Icon,
  default960 as PaintbrushIcon,
  default1741 as PaintbrushVerticalIcon,
  default961 as PaletteIcon,
  default1920 as PalmtreeIcon,
  default962 as PandaIcon,
  default963 as PanelBottomCloseIcon,
  default1743 as PanelBottomDashedIcon,
  default964 as PanelBottomIcon,
  default1744 as PanelBottomInactiveIcon,
  default965 as PanelBottomOpenIcon,
  default1745 as PanelLeftCloseIcon,
  default1747 as PanelLeftDashedIcon,
  default1753 as PanelLeftIcon,
  default1748 as PanelLeftInactiveIcon,
  default1749 as PanelLeftOpenIcon,
  default966 as PanelLeftRightDashedIcon,
  default967 as PanelRightCloseIcon,
  default1751 as PanelRightDashedIcon,
  default969 as PanelRightIcon,
  default1752 as PanelRightInactiveIcon,
  default968 as PanelRightOpenIcon,
  default970 as PanelTopBottomDashedIcon,
  default971 as PanelTopCloseIcon,
  default1755 as PanelTopDashedIcon,
  default973 as PanelTopIcon,
  default1756 as PanelTopInactiveIcon,
  default972 as PanelTopOpenIcon,
  default974 as PanelsLeftBottomIcon,
  default1614 as PanelsLeftRightIcon,
  default975 as PanelsRightBottomIcon,
  default1770 as PanelsTopBottomIcon,
  default1757 as PanelsTopLeftIcon,
  default976 as PaperclipIcon,
  default977 as ParenthesesIcon,
  default1576 as ParkingCircleIcon,
  default1572 as ParkingCircleOffIcon,
  default978 as ParkingMeterIcon,
  default1865 as ParkingSquareIcon,
  default1861 as ParkingSquareOffIcon,
  default979 as PartyPopperIcon,
  default1574 as PauseCircleIcon,
  default981 as PauseIcon,
  default1738 as PauseOctagonIcon,
  default980 as PawPrintIcon,
  default982 as PcCaseIcon,
  default1867 as PenBoxIcon,
  default1761 as PenIcon,
  default1759 as PenLineIcon,
  default983 as PenOffIcon,
  default1869 as PenSquareIcon,
  default984 as PenToolIcon,
  default988 as PencilIcon,
  default986 as PencilLineIcon,
  default985 as PencilOffIcon,
  default987 as PencilRulerIcon,
  default989 as PentagonIcon,
  default1578 as PercentCircleIcon,
  default1618 as PercentDiamondIcon,
  default991 as PercentIcon,
  default1873 as PercentSquareIcon,
  default990 as PersonStandingIcon,
  default992 as PhilippinePesoIcon,
  default993 as PhoneCallIcon,
  default994 as PhoneForwardedIcon,
  default999 as PhoneIcon,
  default995 as PhoneIncomingIcon,
  default996 as PhoneMissedIcon,
  default998 as PhoneOffIcon,
  default997 as PhoneOutgoingIcon,
  default1000 as PiIcon,
  default1871 as PiSquareIcon,
  default1001 as PianoIcon,
  default1002 as PickaxeIcon,
  default1004 as PictureInPicture2Icon,
  default1003 as PictureInPictureIcon,
  default1530 as PieChartIcon,
  default1005 as PiggyBankIcon,
  default1008 as PilcrowIcon,
  default1006 as PilcrowLeftIcon,
  default1007 as PilcrowRightIcon,
  default1875 as PilcrowSquareIcon,
  default1010 as PillBottleIcon,
  default1009 as PillIcon,
  default1012 as PinIcon,
  default1011 as PinOffIcon,
  default1014 as PipetteIcon,
  default1013 as PizzaIcon,
  default1017 as PlaneIcon,
  default1015 as PlaneLandingIcon,
  default1016 as PlaneTakeoffIcon,
  default1580 as PlayCircleIcon,
  default1018 as PlayIcon,
  default1877 as PlaySquareIcon,
  default1019 as Plug2Icon,
  default1020 as PlugIcon,
  default1764 as PlugZap2Icon,
  default1763 as PlugZapIcon,
  default1582 as PlusCircleIcon,
  default1021 as PlusIcon,
  default1883 as PlusSquareIcon,
  default1023 as PocketIcon,
  default1022 as PocketKnifeIcon,
  default1024 as PodcastIcon,
  default1027 as PointerIcon,
  default1025 as PointerOffIcon,
  default1026 as PopcornIcon,
  default1028 as PopsicleIcon,
  default1029 as PoundSterlingIcon,
  default1587 as PowerCircleIcon,
  default1031 as PowerIcon,
  default1030 as PowerOffIcon,
  default1879 as PowerSquareIcon,
  default1034 as PresentationIcon,
  default1032 as PrinterCheckIcon,
  default1035 as PrinterIcon,
  default1033 as PrinterXIcon,
  default1036 as ProjectorIcon,
  default1037 as ProportionsIcon,
  default1038 as PuzzleIcon,
  default1039 as PyramidIcon,
  default1042 as QrCodeIcon,
  default1040 as QuoteIcon,
  default1043 as RabbitIcon,
  default1041 as RadarIcon,
  default1044 as RadiationIcon,
  default1045 as RadicalIcon,
  default1048 as RadioIcon,
  default1046 as RadioReceiverIcon,
  default1047 as RadioTowerIcon,
  default1049 as RadiusIcon,
  default1051 as RailSymbolIcon,
  default1050 as RainbowIcon,
  default1052 as RatIcon,
  default1054 as RatioIcon,
  default1053 as ReceiptCentIcon,
  default1055 as ReceiptEuroIcon,
  default1062 as ReceiptIcon,
  default1056 as ReceiptIndianRupeeIcon,
  default1058 as ReceiptJapaneseYenIcon,
  default1057 as ReceiptPoundSterlingIcon,
  default1059 as ReceiptRussianRubleIcon,
  default1063 as ReceiptSwissFrancIcon,
  default1061 as ReceiptTextIcon,
  default1060 as ReceiptTurkishLiraIcon,
  default1064 as RectangleCircleIcon,
  default1765 as RectangleEllipsisIcon,
  default1065 as RectangleGogglesIcon,
  default1067 as RectangleHorizontalIcon,
  default1068 as RectangleVerticalIcon,
  default1066 as RecycleIcon,
  default1069 as Redo2Icon,
  default1070 as RedoDotIcon,
  default1072 as RedoIcon,
  default1071 as RefreshCcwDotIcon,
  default1074 as RefreshCcwIcon,
  default1075 as RefreshCwIcon,
  default1073 as RefreshCwOffIcon,
  default1078 as RefrigeratorIcon,
  default1076 as RegexIcon,
  default1077 as RemoveFormattingIcon,
  default1079 as Repeat1Icon,
  default1080 as Repeat2Icon,
  default1081 as RepeatIcon,
  default1082 as ReplaceAllIcon,
  default1083 as ReplaceIcon,
  default1086 as ReplyAllIcon,
  default1084 as ReplyIcon,
  default1085 as RewindIcon,
  default1087 as RibbonIcon,
  default1088 as RocketIcon,
  default1090 as RockingChairIcon,
  default1089 as RollerCoasterIcon,
  default1091 as RoseIcon,
  default1768 as Rotate3DIcon,
  default1767 as Rotate3dIcon,
  default1095 as RotateCcwIcon,
  default1092 as RotateCcwKeyIcon,
  default1093 as RotateCcwSquareIcon,
  default1096 as RotateCwIcon,
  default1094 as RotateCwSquareIcon,
  default1098 as RouteIcon,
  default1097 as RouteOffIcon,
  default1099 as RouterIcon,
  default1771 as Rows2Icon,
  default1769 as Rows3Icon,
  default1101 as Rows4Icon,
  default1772 as RowsIcon,
  default1100 as RssIcon,
  default1102 as RulerDimensionLineIcon,
  default1103 as RulerIcon,
  default1104 as RussianRubleIcon,
  default1105 as SailboatIcon,
  default1106 as SaladIcon,
  default1107 as SandwichIcon,
  default1108 as SatelliteDishIcon,
  default1109 as SatelliteIcon,
  default1110 as SaudiRiyalIcon,
  default1111 as SaveAllIcon,
  default1112 as SaveIcon,
  default1113 as SaveOffIcon,
  default1774 as Scale3DIcon,
  default1773 as Scale3dIcon,
  default1114 as ScaleIcon,
  default1115 as ScalingIcon,
  default1116 as ScanBarcodeIcon,
  default1117 as ScanEyeIcon,
  default1118 as ScanFaceIcon,
  default1119 as ScanHeartIcon,
  default1124 as ScanIcon,
  default1120 as ScanLineIcon,
  default1121 as ScanQrCodeIcon,
  default1122 as ScanSearchIcon,
  default1123 as ScanTextIcon,
  default1532 as ScatterChartIcon,
  default1926 as School2Icon,
  default1125 as SchoolIcon,
  default1126 as ScissorsIcon,
  default1127 as ScissorsLineDashedIcon,
  default1816 as ScissorsSquareDashedBottomIcon,
  default1881 as ScissorsSquareIcon,
  default1128 as ScooterIcon,
  default1129 as ScreenShareIcon,
  default1130 as ScreenShareOffIcon,
  default1132 as ScrollIcon,
  default1131 as ScrollTextIcon,
  default1133 as SearchAlertIcon,
  default1134 as SearchCheckIcon,
  default1135 as SearchCodeIcon,
  default1137 as SearchIcon,
  default1136 as SearchSlashIcon,
  default1138 as SearchXIcon,
  default1139 as SectionIcon,
  default1776 as SendHorizonalIcon,
  default1775 as SendHorizontalIcon,
  default1141 as SendIcon,
  default1140 as SendToBackIcon,
  default1142 as SeparatorHorizontalIcon,
  default1143 as SeparatorVerticalIcon,
  default1144 as ServerCogIcon,
  default1145 as ServerCrashIcon,
  default1147 as ServerIcon,
  default1146 as ServerOffIcon,
  default1148 as Settings2Icon,
  default1150 as SettingsIcon,
  default1149 as ShapesIcon,
  default1151 as Share2Icon,
  default1152 as ShareIcon,
  default1153 as SheetIcon,
  default1154 as ShellIcon,
  default1155 as ShelvingUnitIcon,
  default1156 as ShieldAlertIcon,
  default1157 as ShieldBanIcon,
  default1158 as ShieldCheckIcon,
  default1780 as ShieldCloseIcon,
  default1159 as ShieldEllipsisIcon,
  default1160 as ShieldHalfIcon,
  default1165 as ShieldIcon,
  default1161 as ShieldMinusIcon,
  default1162 as ShieldOffIcon,
  default1164 as ShieldPlusIcon,
  default1778 as ShieldQuestionIcon,
  default1777 as ShieldQuestionMarkIcon,
  default1163 as ShieldUserIcon,
  default1779 as ShieldXIcon,
  default1168 as ShipIcon,
  default1166 as ShipWheelIcon,
  default1167 as ShirtIcon,
  default1169 as ShoppingBagIcon,
  default1171 as ShoppingBasketIcon,
  default1170 as ShoppingCartIcon,
  default1172 as ShovelIcon,
  default1173 as ShowerHeadIcon,
  default1174 as ShredderIcon,
  default1175 as ShrimpIcon,
  default1177 as ShrinkIcon,
  default1176 as ShrubIcon,
  default1178 as ShuffleIcon,
  default1746 as SidebarCloseIcon,
  default1754 as SidebarIcon,
  default1750 as SidebarOpenIcon,
  default1179 as SigmaIcon,
  default1885 as SigmaSquareIcon,
  default1180 as SignalHighIcon,
  default1183 as SignalIcon,
  default1181 as SignalLowIcon,
  default1182 as SignalMediumIcon,
  default1184 as SignalZeroIcon,
  default1185 as SignatureIcon,
  default1186 as SignpostBigIcon,
  default1187 as SignpostIcon,
  default1188 as SirenIcon,
  default1191 as SkipBackIcon,
  default1189 as SkipForwardIcon,
  default1190 as SkullIcon,
  default1192 as SlackIcon,
  default1193 as SlashIcon,
  default1889 as SlashSquareIcon,
  default1194 as SliceIcon,
  default1195 as SlidersHorizontalIcon,
  default1782 as SlidersIcon,
  default1781 as SlidersVerticalIcon,
  default1197 as SmartphoneChargingIcon,
  default1198 as SmartphoneIcon,
  default1196 as SmartphoneNfcIcon,
  default1200 as SmileIcon,
  default1199 as SmilePlusIcon,
  default1201 as SnailIcon,
  default1202 as SnowflakeIcon,
  default1203 as SoapDispenserDropletIcon,
  default1204 as SofaIcon,
  default1205 as SolarPanelIcon,
  default1488 as SortAscIcon,
  default1482 as SortDescIcon,
  default1206 as SoupIcon,
  default1207 as SpaceIcon,
  default1209 as SpadeIcon,
  default1208 as SparkleIcon,
  default1783 as SparklesIcon,
  default1210 as SpeakerIcon,
  default1211 as SpeechIcon,
  default1212 as SpellCheck2Icon,
  default1213 as SpellCheckIcon,
  default1216 as SplineIcon,
  default1214 as SplinePointerIcon,
  default1215 as SplitIcon,
  default1887 as SplitSquareHorizontalIcon,
  default1891 as SplitSquareVerticalIcon,
  default1217 as SpoolIcon,
  default1218 as SpotlightIcon,
  default1220 as SprayCanIcon,
  default1219 as SproutIcon,
  default1785 as SquareActivityIcon,
  default1795 as SquareArrowDownIcon,
  default1787 as SquareArrowDownLeftIcon,
  default1789 as SquareArrowDownRightIcon,
  default1791 as SquareArrowLeftIcon,
  default1793 as SquareArrowOutDownLeftIcon,
  default1799 as SquareArrowOutDownRightIcon,
  default1797 as SquareArrowOutUpLeftIcon,
  default1801 as SquareArrowOutUpRightIcon,
  default1221 as SquareArrowRightEnterIcon,
  default1222 as SquareArrowRightExitIcon,
  default1803 as SquareArrowRightIcon,
  default1811 as SquareArrowUpIcon,
  default1805 as SquareArrowUpLeftIcon,
  default1807 as SquareArrowUpRightIcon,
  default1809 as SquareAsteriskIcon,
  default1815 as SquareBottomDashedScissorsIcon,
  default1813 as SquareCenterlineDashedHorizontalIcon,
  default1817 as SquareCenterlineDashedVerticalIcon,
  default1821 as SquareChartGanttIcon,
  default1819 as SquareCheckBigIcon,
  default1824 as SquareCheckIcon,
  default1828 as SquareChevronDownIcon,
  default1826 as SquareChevronLeftIcon,
  default1830 as SquareChevronRightIcon,
  default1832 as SquareChevronUpIcon,
  default1834 as SquareCodeIcon,
  default1223 as SquareDashedBottomCodeIcon,
  default1224 as SquareDashedBottomIcon,
  default1840 as SquareDashedIcon,
  default1836 as SquareDashedKanbanIcon,
  default1838 as SquareDashedMousePointerIcon,
  default1225 as SquareDashedTopSolidIcon,
  default1842 as SquareDivideIcon,
  default1844 as SquareDotIcon,
  default1846 as SquareEqualIcon,
  default1848 as SquareFunctionIcon,
  default1823 as SquareGanttChartIcon,
  default1233 as SquareIcon,
  default1850 as SquareKanbanIcon,
  default1852 as SquareLibraryIcon,
  default1854 as SquareMIcon,
  default1856 as SquareMenuIcon,
  default1862 as SquareMinusIcon,
  default1858 as SquareMousePointerIcon,
  default1864 as SquareParkingIcon,
  default1860 as SquareParkingOffIcon,
  default1226 as SquarePauseIcon,
  default1866 as SquarePenIcon,
  default1872 as SquarePercentIcon,
  default1870 as SquarePiIcon,
  default1874 as SquarePilcrowIcon,
  default1876 as SquarePlayIcon,
  default1882 as SquarePlusIcon,
  default1878 as SquarePowerIcon,
  default1227 as SquareRadicalIcon,
  default1228 as SquareRoundCornerIcon,
  default1880 as SquareScissorsIcon,
  default1884 as SquareSigmaIcon,
  default1888 as SquareSlashIcon,
  default1886 as SquareSplitHorizontalIcon,
  default1890 as SquareSplitVerticalIcon,
  default1229 as SquareSquareIcon,
  default1230 as SquareStackIcon,
  default1232 as SquareStarIcon,
  default1231 as SquareStopIcon,
  default1896 as SquareTerminalIcon,
  default1894 as SquareUserIcon,
  default1892 as SquareUserRoundIcon,
  default1898 as SquareXIcon,
  default1234 as SquaresExcludeIcon,
  default1235 as SquaresIntersectIcon,
  default1237 as SquaresSubtractIcon,
  default1236 as SquaresUniteIcon,
  default1239 as SquircleDashedIcon,
  default1238 as SquircleIcon,
  default1240 as SquirrelIcon,
  default1241 as StampIcon,
  default1242 as StarHalfIcon,
  default1245 as StarIcon,
  default1243 as StarOffIcon,
  default1784 as StarsIcon,
  default1244 as StepBackIcon,
  default1246 as StepForwardIcon,
  default1247 as StethoscopeIcon,
  default1248 as StickerIcon,
  default1249 as StickyNoteIcon,
  default1250 as StoneIcon,
  default1591 as StopCircleIcon,
  default1251 as StoreIcon,
  default1252 as StretchHorizontalIcon,
  default1253 as StretchVerticalIcon,
  default1254 as StrikethroughIcon,
  default1255 as SubscriptIcon,
  default1506 as SubtitlesIcon,
  default1256 as SunDimIcon,
  default1259 as SunIcon,
  default1257 as SunMediumIcon,
  default1258 as SunMoonIcon,
  default1260 as SunSnowIcon,
  default1261 as SunriseIcon,
  default1262 as SunsetIcon,
  default1263 as SuperscriptIcon,
  default1264 as SwatchBookIcon,
  default1265 as SwissFrancIcon,
  default1267 as SwitchCameraIcon,
  default1266 as SwordIcon,
  default1269 as SwordsIcon,
  default1268 as SyringeIcon,
  default1271 as Table2Icon,
  default1270 as TableCellsMergeIcon,
  default1272 as TableCellsSplitIcon,
  default1273 as TableColumnsSplitIcon,
  default1612 as TableConfigIcon,
  default1276 as TableIcon,
  default1274 as TableOfContentsIcon,
  default1275 as TablePropertiesIcon,
  default1278 as TableRowsSplitIcon,
  default1279 as TabletIcon,
  default1277 as TabletSmartphoneIcon,
  default1280 as TabletsIcon,
  default1281 as TagIcon,
  default1283 as TagsIcon,
  default1282 as Tally1Icon,
  default1285 as Tally2Icon,
  default1284 as Tally3Icon,
  default1286 as Tally4Icon,
  default1287 as Tally5Icon,
  default1288 as TangentIcon,
  default1289 as TargetIcon,
  default1290 as TelescopeIcon,
  default1292 as TentIcon,
  default1291 as TentTreeIcon,
  default1293 as TerminalIcon,
  default1897 as TerminalSquareIcon,
  default1901 as TestTube2Icon,
  default1900 as TestTubeDiagonalIcon,
  default1294 as TestTubeIcon,
  default1295 as TestTubesIcon,
  default1902 as TextAlignCenterIcon,
  default1904 as TextAlignEndIcon,
  default1906 as TextAlignJustifyIcon,
  default1908 as TextAlignStartIcon,
  default1297 as TextCursorIcon,
  default1296 as TextCursorInputIcon,
  default1909 as TextIcon,
  default1911 as TextInitialIcon,
  default1298 as TextQuoteIcon,
  default1299 as TextSearchIcon,
  default1913 as TextSelectIcon,
  default1914 as TextSelectionIcon,
  default1915 as TextWrapIcon,
  default1300 as TheaterIcon,
  default1302 as ThermometerIcon,
  default1301 as ThermometerSnowflakeIcon,
  default1303 as ThermometerSunIcon,
  default1304 as ThumbsDownIcon,
  default1306 as ThumbsUpIcon,
  default1305 as TicketCheckIcon,
  default1313 as TicketIcon,
  default1307 as TicketMinusIcon,
  default1308 as TicketPercentIcon,
  default1309 as TicketPlusIcon,
  default1310 as TicketSlashIcon,
  default1311 as TicketXIcon,
  default1314 as TicketsIcon,
  default1312 as TicketsPlaneIcon,
  default1317 as TimerIcon,
  default1315 as TimerOffIcon,
  default1316 as TimerResetIcon,
  default1318 as ToggleLeftIcon,
  default1319 as ToggleRightIcon,
  default1320 as ToiletIcon,
  default1321 as ToolCaseIcon,
  default1322 as ToolboxIcon,
  default1323 as TornadoIcon,
  default1324 as TorusIcon,
  default1327 as TouchpadIcon,
  default1325 as TouchpadOffIcon,
  default1326 as TowelRackIcon,
  default1328 as TowerControlIcon,
  default1329 as ToyBrickIcon,
  default1330 as TractorIcon,
  default1331 as TrafficConeIcon,
  default1335 as TrainFrontIcon,
  default1332 as TrainFrontTunnelIcon,
  default1918 as TrainIcon,
  default1333 as TrainTrackIcon,
  default1917 as TramFrontIcon,
  default1336 as TransgenderIcon,
  default1334 as Trash2Icon,
  default1337 as TrashIcon,
  default1338 as TreeDeciduousIcon,
  default1919 as TreePalmIcon,
  default1339 as TreePineIcon,
  default1340 as TreesIcon,
  default1341 as TrelloIcon,
  default1342 as TrendingDownIcon,
  default1343 as TrendingUpDownIcon,
  default1346 as TrendingUpIcon,
  default1921 as TriangleAlertIcon,
  default1345 as TriangleDashedIcon,
  default1349 as TriangleIcon,
  default1344 as TriangleRightIcon,
  default1348 as TrophyIcon,
  default1347 as TruckElectricIcon,
  default1350 as TruckIcon,
  default1351 as TurkishLiraIcon,
  default1352 as TurntableIcon,
  default1353 as TurtleIcon,
  default1924 as Tv2Icon,
  default1356 as TvIcon,
  default1923 as TvMinimalIcon,
  default1354 as TvMinimalPlayIcon,
  default1355 as TwitchIcon,
  default1357 as TwitterIcon,
  default1359 as TypeIcon,
  default1358 as TypeOutlineIcon,
  default1361 as UmbrellaIcon,
  default1360 as UmbrellaOffIcon,
  default1362 as UnderlineIcon,
  default1363 as Undo2Icon,
  default1364 as UndoDotIcon,
  default1365 as UndoIcon,
  default1367 as UnfoldHorizontalIcon,
  default1366 as UnfoldVerticalIcon,
  default1368 as UngroupIcon,
  default1925 as UniversityIcon,
  default1369 as Unlink2Icon,
  default1370 as UnlinkIcon,
  default1726 as UnlockIcon,
  default1722 as UnlockKeyholeIcon,
  default1371 as UnplugIcon,
  default1605 as UploadCloudIcon,
  default1373 as UploadIcon,
  default1372 as UsbIcon,
  default1938 as User2Icon,
  default1928 as UserCheck2Icon,
  default1374 as UserCheckIcon,
  default1595 as UserCircle2Icon,
  default1593 as UserCircleIcon,
  default1930 as UserCog2Icon,
  default1376 as UserCogIcon,
  default1387 as UserIcon,
  default1375 as UserKeyIcon,
  default1377 as UserLockIcon,
  default1932 as UserMinus2Icon,
  default1379 as UserMinusIcon,
  default1378 as UserPenIcon,
  default1934 as UserPlus2Icon,
  default1380 as UserPlusIcon,
  default1927 as UserRoundCheckIcon,
  default1929 as UserRoundCogIcon,
  default1937 as UserRoundIcon,
  default1381 as UserRoundKeyIcon,
  default1931 as UserRoundMinusIcon,
  default1382 as UserRoundPenIcon,
  default1933 as UserRoundPlusIcon,
  default1383 as UserRoundSearchIcon,
  default1935 as UserRoundXIcon,
  default1384 as UserSearchIcon,
  default1893 as UserSquare2Icon,
  default1895 as UserSquareIcon,
  default1385 as UserStarIcon,
  default1936 as UserX2Icon,
  default1386 as UserXIcon,
  default1940 as Users2Icon,
  default1388 as UsersIcon,
  default1939 as UsersRoundIcon,
  default1943 as UtensilsCrossedIcon,
  default1941 as UtensilsIcon,
  default1389 as UtilityPoleIcon,
  default1390 as VanIcon,
  default1392 as VariableIcon,
  default1391 as VaultIcon,
  default1393 as VectorSquareIcon,
  default1395 as VeganIcon,
  default1394 as VenetianMaskIcon,
  default1397 as VenusAndMarsIcon,
  default1396 as VenusIcon,
  default1494 as VerifiedIcon,
  default1399 as VibrateIcon,
  default1398 as VibrateOffIcon,
  default1402 as VideoIcon,
  default1400 as VideoOffIcon,
  default1401 as VideotapeIcon,
  default1403 as ViewIcon,
  default1404 as VoicemailIcon,
  default1405 as VolleyballIcon,
  default1406 as Volume1Icon,
  default1407 as Volume2Icon,
  default1410 as VolumeIcon,
  default1408 as VolumeOffIcon,
  default1409 as VolumeXIcon,
  default1411 as VoteIcon,
  default1948 as Wallet2Icon,
  default1412 as WalletCardsIcon,
  default1413 as WalletIcon,
  default1947 as WalletMinimalIcon,
  default1414 as WallpaperIcon,
  default1946 as Wand2Icon,
  default1415 as WandIcon,
  default1945 as WandSparklesIcon,
  default1416 as WarehouseIcon,
  default1417 as WashingMachineIcon,
  default1418 as WatchIcon,
  default1419 as WavesArrowDownIcon,
  default1420 as WavesArrowUpIcon,
  default1422 as WavesIcon,
  default1421 as WavesLadderIcon,
  default1423 as WaypointsIcon,
  default1424 as WebcamIcon,
  default1427 as WebhookIcon,
  default1425 as WebhookOffIcon,
  default1428 as WeightIcon,
  default1426 as WeightTildeIcon,
  default1429 as WheatIcon,
  default1430 as WheatOffIcon,
  default1431 as WholeWordIcon,
  default1432 as WifiCogIcon,
  default1433 as WifiHighIcon,
  default1439 as WifiIcon,
  default1434 as WifiLowIcon,
  default1435 as WifiOffIcon,
  default1437 as WifiPenIcon,
  default1438 as WifiSyncIcon,
  default1436 as WifiZeroIcon,
  default1440 as WindArrowDownIcon,
  default1443 as WindIcon,
  default1442 as WineIcon,
  default1441 as WineOffIcon,
  default1445 as WorkflowIcon,
  default1444 as WormIcon,
  default1916 as WrapTextIcon,
  default1446 as WrenchIcon,
  default1597 as XCircleIcon,
  default1448 as XIcon,
  default1447 as XLineTopIcon,
  default1740 as XOctagonIcon,
  default1899 as XSquareIcon,
  default1449 as YoutubeIcon,
  default1451 as ZapIcon,
  default1450 as ZapOffIcon,
  default1452 as ZodiacAquariusIcon,
  default1453 as ZodiacAriesIcon,
  default1455 as ZodiacCancerIcon,
  default1454 as ZodiacCapricornIcon,
  default1457 as ZodiacGeminiIcon,
  default1456 as ZodiacLeoIcon,
  default1458 as ZodiacLibraIcon,
  default1460 as ZodiacOphiuchusIcon,
  default1459 as ZodiacPiscesIcon,
  default1461 as ZodiacSagittariusIcon,
  default1462 as ZodiacScorpioIcon,
  default1464 as ZodiacTaurusIcon,
  default1463 as ZodiacVirgoIcon,
  default1465 as ZoomInIcon,
  default1466 as ZoomOutIcon
};
